--- content_archive/IPA_what-are-tradmarks.md	2026-02-12 19:00:06.042360075 +0000
+++ content_archive/IPA_what-are-tradmarks.md.tmp	2026-02-12 19:00:36.364136928 +0000
@@ -154,11 +154,13 @@
 [Types of trade marks](/trade-marks/what-are-trade-marks/kinds-of-trade-marks)
 
 ![Three books with red shading, and a single exclamation mark down the book spine](/-/media/Project/IPA/IPAustralia/illustrations/trade-marks/categories.png?h=416&iar=0&w=416&rev=f30f9ba872e84bf19262a94bd8d2c281&hash=27DD64ED02316D6BD83037505B93BB83)
 
 Our website uses cookies to improve your experience.
 
 [Learn more about how we use and collect data](/about-us/our-agency/privacy)
 
 Accept
 
-Opt Out
\ No newline at end of file
+Opt Out
+
+Skip embedded contentReturn to page content
\ No newline at end of file
