--- content_archive/TradeMarksAct1995.md	2026-02-20 03:22:31.286752770 +0000
+++ content_archive/TradeMarksAct1995.md.tmp	2026-02-20 03:23:16.495533529 +0000
@@ -1,11 +1,11 @@
-Trade Marks Act 1995 JENN WAS HERE
+Trade Marks Act 1995
 
 No. 119, 1995
 
 Compilation No. 47
 
 Compilation date:	14 October 2024
 
 Includes amendments:	Act No. 39, 2024
 
 About this compilation
@@ -6717,11 +6717,11 @@
 Misdescribed amendments
 
 A misdescribed amendment is an amendment that does not accurately describe how an amendment is to be made. If, despite the misdescription, the amendment can be given effect as intended, then the misdescribed amendment can be incorporated through an editorial change made under section 15V of the Legislation Act 2003.
 
 If a misdescribed amendment cannot be given effect as intended, the amendment is not incorporated and “(md not incorp)” is added to the amendment history.
 
 Endnote 2—Abbreviation key
 
 Endnote 3—Legislation history
 
-Endnote 4—Amendment history
+Endnote 4—Amendment history
\ No newline at end of file
