--- content_archive/ABC_News.xml	2026-02-08 02:24:50.186227055 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-08 02:25:21.276185773 +0000
@@ -32,153 +32,153 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    How Epstein's ties to the 'Prince of Darkness' could bring down Britain's PM
+    'Monster' funeral home owner sentenced over 189 counts of corpse abuse
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-07/starmer-uk-pm-mandelson-epstein-files-calls-to-resign/106315470
+    https://www.abc.net.au/news/2026-02-07/colorado-funeral-home-owner-sentenced-over-corpse-abuse/106317582
    </link>
    <description>
-    Sir Keir Starmer led Labour to a landslide election victory two years ago, but in the past week, things have unravelled for the British prime minister. What went wrong?
+    A funeral home owner who stashed 189 decomposing bodies in a building over four years and gave grieving families fake ashes has been sentenced to 40 years in prison.
    </description>
-   <dc:creator>
-    Cason Ho
-   </dc:creator>
    <pubDate>
-    Fri, 06 Feb 2026 19:17:21 +0000
+    Sat, 07 Feb 2026 05:36:16 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-07/starmer-uk-pm-mandelson-epstein-files-calls-to-resign/106315470
+    https://www.abc.net.au/news/2026-02-07/colorado-funeral-home-owner-sentenced-over-corpse-abuse/106317582
    </guid>
    <category>
-    World Politics
-   </category>
-   <category>
-    Sexual Offences
+    Courts
    </category>
    <media:group>
     <media:description>
-     Former British ambassador to the US Peter Mandelson's (left) relationship with Jeffrey Epstein has created a pressure cooker for British Prime Minister Keir Starmer who appointed him.
+     Family members of the victims have labelled the funeral home owner a "monster".
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=260&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=3333&amp;xPos=834&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=4444&amp;xPos=278&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=1875&amp;xPos=1563&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/d9453900647b3fe43a18b0d319363a87?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=1688&amp;cropW=3000&amp;xPos=0&amp;yPos=102&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=2075&amp;cropW=2075&amp;xPos=472&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=3000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=2075&amp;cropW=1556&amp;xPos=732&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=2075&amp;cropW=2767&amp;xPos=126&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=2075&amp;cropW=1167&amp;xPos=926&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/40243963fb96abe7411e225ab96ca18d?impolicy=wcms_crop_resize&amp;cropH=2075&amp;cropW=1556&amp;xPos=732&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Trump endorses Japan's prime minister ahead of Sunday election
+    'You always make me smile': Epstein files rock global monarchies
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-07/trump-endorses-japan-pm-takaichi-ahead-of-sunday-election/106317102
+    https://www.abc.net.au/news/2026-02-07/epstein-files-rock-global-monarchies/106316380
    </link>
    <description>
-    One of Sanae Takaichi's first engagements after she was elevated to prime minister in October was to host Donald Trump in Tokyo.
+    The latest release of the so-called Epstein files has revealed the convicted child sex offender was in contact with members of — and people close to — monarchies across Europe and the Middle East.
    </description>
+   <dc:creator>
+    Syan Vallance
+   </dc:creator>
    <pubDate>
-    Fri, 06 Feb 2026 23:05:47 +0000
+    Sat, 07 Feb 2026 04:10:33 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-07/trump-endorses-japan-pm-takaichi-ahead-of-sunday-election/106317102
+    https://www.abc.net.au/news/2026-02-07/epstein-files-rock-global-monarchies/106316380
    </guid>
    <category>
-    Elections
+    Royalty
    </category>
    <category>
-    World Politics
+    Law, Crime and Justice
    </category>
    <media:group>
     <media:description>
-     Analysts say Donald Trump's endorsement of Sanae Takaichi signals a growing pattern of aligning with right-wing leaders abroad.
+     Convicted child sex offender Jeffrey Epstein died by suicide behind bars in 2019.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2283&amp;cropW=4059&amp;xPos=0&amp;yPos=208&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2843&amp;cropW=2843&amp;xPos=294&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2706&amp;cropW=4059&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2843&amp;cropW=2132&amp;xPos=650&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2843&amp;cropW=3791&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2843&amp;cropW=1599&amp;xPos=916&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/12ea931c36264548108303ab82e22542?impolicy=wcms_crop_resize&amp;cropH=2843&amp;cropW=2132&amp;xPos=650&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1120&amp;cropW=1989&amp;xPos=5&amp;yPos=291&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1690&amp;cropW=1690&amp;xPos=155&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1331&amp;cropW=2000&amp;xPos=0&amp;yPos=160&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1690&amp;cropW=1266&amp;xPos=367&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1497&amp;cropW=2000&amp;xPos=0&amp;yPos=40&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1690&amp;cropW=950&amp;xPos=525&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/d1c40893c7ccd2ed5608e4d53a0d77df?impolicy=wcms_crop_resize&amp;cropH=1690&amp;cropW=1266&amp;xPos=367&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    US accuses China of secret nuclear tests as calls grow for new arms deal
+    Americans propose an end to Russian-Ukrainian war by June, Zelenskyy says
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-07/us-accuses-china-of-secret-nuclear-testing/106316826
+    https://www.abc.net.au/news/2026-02-07/russia-ukraine-deadline-to-end-war-/106317976
    </link>
    <description>
-    The claim comes as the US calls for China to be involved in a new deal limiting the use of nuclear weapons, after the New START deal expired this week.
+    America has given Russia and Ukraine a June deadline to end the war — and will put pressure on both sides to ensure it is met — Volodymyr Zelenskyy says.
    </description>
    <pubDate>
-    Fri, 06 Feb 2026 16:14:30 +0000
+    Sat, 07 Feb 2026 12:17:54 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-07/us-accuses-china-of-secret-nuclear-testing/106316826
+    https://www.abc.net.au/news/2026-02-07/russia-ukraine-deadline-to-end-war-/106317976
    </guid>
    <category>
-    Nuclear Weapons
+    Unrest, Conflict and War
+   </category>
+   <category>
+    World Politics
    </category>
    <media:group>
     <media:description>
-     A ground-based intercontinental ballistic missile test at the Plesetsk facility in north-western Russia.
+     Volodymyr Zelenskyy says the US has given Ukraine and Russia a June deadline to end the war.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2350&amp;cropW=4178&amp;xPos=0&amp;yPos=1&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2352&amp;cropW=2352&amp;xPos=913&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2352&amp;cropW=3528&amp;xPos=325&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2352&amp;cropW=1764&amp;xPos=1207&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2352&amp;cropW=3136&amp;xPos=521&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2352&amp;cropW=1323&amp;xPos=1428&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/afe304ecf3685136b7a0a81bc8bc0ddf?impolicy=wcms_crop_resize&amp;cropH=2352&amp;cropW=1764&amp;xPos=1207&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=260&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=3333&amp;xPos=834&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=4444&amp;xPos=278&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=1875&amp;xPos=1563&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Jeers, cheers and pleas for peace kick off Milano Cortina 2026
+    'They're afraid': Can Thailand's popular progressives finally win government?
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-07/winter-olympics-2026-opening-ceremony-chilly-global-outlook/106316092
+    https://www.abc.net.au/news/2026-02-07/thailand-election-stakes-high-as-progressives-mount-challenge/106312802
    </link>
    <description>
-    As the Milano Cortina 2026 Winter Olympics Opening ceremony pleads a message of peace, the world outside the Olympic bubble burns.
+    As Thailand prepares to head to the polls, voters remain wary that in their version of democracy, the party they vote for is not always the one that takes power.
    </description>
    <dc:creator>
-    Simon Smale
+    Karishma Vyas
    </dc:creator>
    <pubDate>
-    Fri, 06 Feb 2026 23:59:27 +0000
+    Sat, 07 Feb 2026 02:33:50 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-07/winter-olympics-2026-opening-ceremony-chilly-global-outlook/106316092
+    https://www.abc.net.au/news/2026-02-07/thailand-election-stakes-high-as-progressives-mount-challenge/106312802
    </guid>
    <category>
-    Winter Olympic Games
+    Elections
    </category>
    <category>
-    Sport
+    World Politics
    </category>
    <media:group>
     <media:description>
-     The 25th Winter Olympic Games have begun.
+     Thanathorn Juangroongruangkit has been barred from politics for 10 years.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=1747&amp;cropW=3106&amp;xPos=261&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=2360&amp;cropW=2360&amp;xPos=617&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=2360&amp;cropW=3540&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=2360&amp;cropW=1770&amp;xPos=930&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=2360&amp;cropW=3147&amp;xPos=201&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=2360&amp;cropW=1328&amp;xPos=1110&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/3f3513f01bd205ce0ae4184f41d68359?impolicy=wcms_crop_resize&amp;cropH=2360&amp;cropW=1770&amp;xPos=930&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=485&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=3750&amp;cropW=3750&amp;xPos=117&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=414&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=3750&amp;cropW=2813&amp;xPos=585&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=3750&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=3750&amp;cropW=2109&amp;xPos=1017&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/a40c1ed04bebe611395ca3039444f69e?impolicy=wcms_crop_resize&amp;cropH=3750&amp;cropW=2813&amp;xPos=585&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
