--- content_archive/ABC_News.xml	2026-02-12 06:58:57.997843351 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-12 06:59:29.162773600 +0000
@@ -123,57 +123,60 @@
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=2245&amp;xPos=873&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=3368&amp;xPos=312&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=1684&amp;xPos=1154&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=2993&amp;xPos=499&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=1262&amp;xPos=1365&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=1684&amp;xPos=1154&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    US officials say cartel drones behind sudden closure of Texas airport
+    James Van Der Beek, star of Dawson's Creek, dies aged 48
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-12/faa-closes-airspace-around-el-paso-airport/106333810
+    https://www.abc.net.au/news/2026-02-12/james-van-der-beek-dawsons-creek-star-obituary/105821848
    </link>
    <description>
-    US officials say a sudden 10-day ban came into effect around El Paso International Airport after cartel drones from nearby Mexico breached American airspace.
+    The American actor was best known for his starring role as Dawson Leery in Dawson's Creek from 1998 to 2003.
    </description>
    <pubDate>
-    Wed, 11 Feb 2026 16:16:41 +0000
+    Wed, 11 Feb 2026 19:57:38 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-12/faa-closes-airspace-around-el-paso-airport/106333810
+    https://www.abc.net.au/news/2026-02-12/james-van-der-beek-dawsons-creek-star-obituary/105821848
    </guid>
    <category>
-    Air Incidents
+    Celebrities
    </category>
    <category>
-    Defence and National Security
+    Actor
    </category>
    <category>
-    World Politics
+    Television
    </category>
    <category>
-    Air Transport Industry
+    Cancer
+   </category>
+   <category>
+    Arts, Culture and Entertainment
    </category>
    <media:group>
     <media:description>
-     Airspace 16 kilometres around El Paso International Airport was closed for several hours until the FAA rescinded its notice.
+     James Van Der Beek starred alongside Katie Holmes in Dawson's Creek.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=4293&amp;xPos=353&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=2415&amp;xPos=1293&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=3623&amp;xPos=689&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=1811&amp;xPos=1594&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=3220&amp;xPos=890&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=1358&amp;xPos=1821&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/4b2e07e2358cf5e014646e8648144d07?impolicy=wcms_crop_resize&amp;cropH=2415&amp;cropW=1811&amp;xPos=1594&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=892&amp;cropW=1586&amp;xPos=74&amp;yPos=101&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=1689&amp;cropW=1689&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=1142&amp;cropW=1713&amp;xPos=0&amp;yPos=101&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=2142&amp;cropW=1606&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=1277&amp;cropW=1703&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=2142&amp;cropW=1205&amp;xPos=359&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/f000c0cf4fa6c4e70c744fa5fed5a898?impolicy=wcms_crop_resize&amp;cropH=2142&amp;cropW=1606&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
     China's president makes rare acknowledgement of corruption in army
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-12/xi-jinping-acknowledges-army-corruption-crackdown/106333834
    </link>
    <description>
