--- content_archive/WIPO_Press.xml+++ new_content@@ -15,26 +15,6 @@   </language>
   <item>
    <title>
-    Investment in Intangible Assets Surges, led by Funding for Software and Databases Amid AI Boom
-   </title>
-   <author>
-    WIPO
-   </author>
-   <link>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0005.html
-   </link>
-   <guid>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0005.html
-   </guid>
-   <description>
-    Investment in intangible assets such as data, software, brands and other intellectual property-backed assets grew three times faster in 2024 than investment in physical assets such as machinery and buildings, which has languished amid high interest rates and subdued economic recovery, according to new data from the World Intellectual Property Organization (WIPO) and Italy’s Luiss Business School.
-   </description>
-   <pubDate>
-    Wed, 09 Jul 2025 08:00:00 +0200
-   </pubDate>
-  </item>
-  <item>
-   <title>
     WIPO Announces 2025 Global Awards Winners: Recognizing IP-Driven Innovation and Impact
    </title>
    <author>
@@ -211,6 +191,26 @@    </description>
    <pubDate>
     Tue, 17 Feb 2026 09:00:00 +0100
+   </pubDate>
+  </item>
+  <item>
+   <title>
+    International Patent Applications Rose in 2025, with Digital Communication and Semiconductor Technologies Showing Strong Growth
+   </title>
+   <author>
+    WIPO
+   </author>
+   <link>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0003.html
+   </link>
+   <guid>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0003.html
+   </guid>
+   <description>
+    International patent applications filed through WIPO's Patent Cooperation Treaty (PCT) rose 0.7% in 2025 to reach 275,900 filings worldwide, marking a second consecutive year of growth. The increase was partly due to robust activity in ICT related fields, with digital communication remaining as the top category (11.1%) of published PCT applications, while semiconductors recorded one of the fastest growth rates among all leading areas of international patents.
+   </description>
+   <pubDate>
+    Fri, 06 Mar 2026 09:30:00 +0100
    </pubDate>
   </item>
  </channel>
