--- content_archive/ABC_News.xml	2026-02-15 02:03:48.439126939 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-15 02:04:28.518166282 +0000
@@ -32,144 +32,168 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    Japan seizes Chinese fishing boat, arrests captain
+    Australia and EU on the cusp of signing landmark free trade agreement
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-13/japan-seizes-chinese-fishing-boat-amid-dispute-between-countries/106343816
+    https://www.abc.net.au/news/2026-02-14/australia-and-eu-sign-landmark-free-trade-agreement/106344826
    </link>
    <description>
-    Japanese authorities have seized a Chinese fishing vessel for the first time since 2022 and it comes against the backdrop of a major diplomatic dispute between the countries.
+    Australia and the European Union are on the cusp of finally reaching a free trade agreement after years of sometimes fraught negotiations.
    </description>
+   <dc:creator>
+    Stephen Dziedzic
+   </dc:creator>
    <pubDate>
-    Fri, 13 Feb 2026 10:35:38 +0000
+    Sat, 14 Feb 2026 00:28:39 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-13/japan-seizes-chinese-fishing-boat-amid-dispute-between-countries/106343816
+    https://www.abc.net.au/news/2026-02-14/australia-and-eu-sign-landmark-free-trade-agreement/106344826
    </guid>
    <category>
+    Trade
+   </category>
+   <category>
+    Federal Government
+   </category>
+   <category>
     Foreign Affairs
    </category>
+   <category>
+    Government and Politics
+   </category>
    <media:group>
     <media:description>
-     Japan's Fisheries Agency patrol vessel Hakuo Maru next to the Chinese vessel Qiong Dong Yu which was seized by Japanese authorities.
+     After days of discussion, Australia is one step closer to closing a deal with the EU.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=1688&amp;cropW=3000&amp;xPos=0&amp;yPos=156&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=2000&amp;xPos=500&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=3000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=1500&amp;xPos=750&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=2667&amp;xPos=167&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=1125&amp;xPos=938&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/cecf02e4d4f08eb159f2e92e5268ac2f?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=1500&amp;xPos=750&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=2812&amp;cropW=5000&amp;xPos=0&amp;yPos=261&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=3335&amp;xPos=832&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=1&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=2501&amp;xPos=2096&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=4447&amp;xPos=277&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=1876&amp;xPos=1562&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=2501&amp;xPos=2096&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    US aircraft carrier redeployed from Caribbean as Iran tensions rise
+    How Indonesians are fighting plummeting marriage rates
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-13/us-aircraft-carrier-carribbean-middle-east-iran/106342830
+    https://www.abc.net.au/news/2026-02-14/indonesian-marriage-rate-plummets-despite-demand/106334832
    </link>
    <description>
-    The United States will send the USS Gerald R. Ford aircraft carrier to the Middle East amid escalating tensions with Iran, according to US media outlets.
+    Marriage is still a life goal in Indonesia, but the number of marriages has plummeted over the past decade.
    </description>
+   <dc:creator>
+    Natasya Salim
+   </dc:creator>
    <pubDate>
-    Fri, 13 Feb 2026 08:14:31 +0000
+    Fri, 13 Feb 2026 21:42:04 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-13/us-aircraft-carrier-carribbean-middle-east-iran/106342830
+    https://www.abc.net.au/news/2026-02-14/indonesian-marriage-rate-plummets-despite-demand/106334832
    </guid>
    <category>
-    Unrest, Conflict and War
+    Marriage
+   </category>
+   <category>
+    Dating
    </category>
    <media:group>
     <media:description>
-     The USS Gerald R. Ford, the world's largest aircraft carrier, is on the move from the Caribbean.
+     Talitha Rarasty (left) works as a makeup artist in Surabaya, East Java, Indonesia.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2382&amp;cropW=4234&amp;xPos=0&amp;yPos=221&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2823&amp;cropW=2823&amp;xPos=706&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2823&amp;cropW=4234&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2823&amp;cropW=2117&amp;xPos=1058&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2823&amp;cropW=3764&amp;xPos=235&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2823&amp;cropW=1588&amp;xPos=1323&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/676dbcfadfb792d73241654225c05e53?impolicy=wcms_crop_resize&amp;cropH=2823&amp;cropW=2117&amp;xPos=1058&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=416&amp;cropW=739&amp;xPos=0&amp;yPos=338&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=739&amp;cropW=739&amp;xPos=0&amp;yPos=167&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=493&amp;cropW=739&amp;xPos=0&amp;yPos=273&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=985&amp;cropW=739&amp;xPos=0&amp;yPos=302&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=554&amp;cropW=739&amp;xPos=0&amp;yPos=232&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=1297&amp;cropW=730&amp;xPos=5&amp;yPos=146&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=985&amp;cropW=739&amp;xPos=0&amp;yPos=302&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Pair who plotted attack on Jews in northern England jailed for life
+    Australian craft brewers hop on 'clean and easy' Japanese lagers
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-14/men-who-plotted-gun-attack-on-uk-jews-jailed-for-life/106344876
+    https://www.abc.net.au/news/2026-02-14/japanese-lagers-thrive-in-australia-after-record-travel-numbers/106119108
    </link>
    <description>
-    A UK court jails two men for life for planning an Islamic State-inspired gun attack on a Jewish gathering in northern England.
+    As record numbers of Australians continue to visit Japan, Japanese brewers have come to dominate our domestic beer market.
    </description>
+   <dc:creator>
+    Max Walden and Lachlan Bennett
+   </dc:creator>
    <pubDate>
-    Sat, 14 Feb 2026 01:38:46 +0000
+    Fri, 13 Feb 2026 19:22:09 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-14/men-who-plotted-gun-attack-on-uk-jews-jailed-for-life/106344876
+    https://www.abc.net.au/news/2026-02-14/japanese-lagers-thrive-in-australia-after-record-travel-numbers/106119108
    </guid>
    <category>
-    Terrorism
+    Travel and Tourism
+   </category>
+   <category>
+    Food and Beverage Processing Industry
    </category>
    <category>
-    Law, Crime and Justice
+    Hospitality Industry
    </category>
    <category>
-    Courts
+    Drinks
    </category>
    <media:group>
     <media:description>
-     Amar Hussein (left) and Walid Saadaoui (right) were arrested in May 2024.
+     Brewers attribute the crisp flavour of Japanese beer to the country's "soft" water.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=960&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=540&amp;xPos=209&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=810&amp;xPos=74&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=405&amp;xPos=277&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=720&amp;xPos=119&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=304&amp;xPos=327&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/3e90c979f214860b7f37216fd373d2d8?impolicy=wcms_crop_resize&amp;cropH=540&amp;cropW=405&amp;xPos=277&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2385&amp;cropW=4240&amp;xPos=0&amp;yPos=7&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=2400&amp;xPos=920&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=3600&amp;xPos=320&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1800&amp;xPos=1220&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=3200&amp;xPos=520&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1350&amp;xPos=1445&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1800&amp;xPos=1220&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Nine people killed in Canada mass shooting identified
+    Russia poisoned Alexei Navalny, UK and allies say
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-14/tumbler-ridge-school-mass-shooting-victims-named/106343974
+    https://www.abc.net.au/news/2026-02-15/russia-poisoned-alexei-navalny-uk-allies-say/106345766
    </link>
    <description>
-    Six children and two adults were killed in the shooting in the remote Canadian town of Tumbler Ridge on Tuesday.
+    Five European foreign ministers say analysis of samples from the former Russian opposition leader, who died two years ago, confirmed the presence of a toxin from poison frogs.
    </description>
    <pubDate>
-    Fri, 13 Feb 2026 13:07:35 +0000
+    Sat, 14 Feb 2026 13:24:14 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-14/tumbler-ridge-school-mass-shooting-victims-named/106343974
+    https://www.abc.net.au/news/2026-02-15/russia-poisoned-alexei-navalny-uk-allies-say/106345766
    </guid>
    <category>
-    Shootings
+    Crime
    </category>
    <media:group>
     <media:description>
-     Abel Mwansa, 12, and Kylie Smith, 12, were among those killed in the mass shooting.
+     Alexei Navalny died in a Russian prison in February 2024.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=450&amp;cropW=800&amp;xPos=0&amp;yPos=64&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=600&amp;cropW=600&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=533&amp;cropW=800&amp;xPos=0&amp;yPos=22&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=600&amp;cropW=450&amp;xPos=180&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=600&amp;cropW=800&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=600&amp;cropW=338&amp;xPos=32&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/d4d06fd879139f1cae9911047ef75244?impolicy=wcms_crop_resize&amp;cropH=600&amp;cropW=450&amp;xPos=180&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=259&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=3330&amp;xPos=835&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=4995&amp;xPos=2&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=2498&amp;xPos=1251&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=4440&amp;xPos=280&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=1873&amp;xPos=1564&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=2498&amp;xPos=1251&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
