--- content_archive/WIPO_Press.xml+++ new_content@@ -15,26 +15,6 @@   </language>
   <item>
    <title>
-    Venture Capital Metric Reshapes WIPO's World’s Top 100 Innovation Clusters Rankings
-   </title>
-   <author>
-    WIPO
-   </author>
-   <link>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0008.html
-   </link>
-   <guid>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0008.html
-   </guid>
-   <description>
-    The addition of venture capital (VC) deal activity as a new metric in this year's WIPO Global Innovation Index (GII) Cluster study has reshaped the ranking with Shenzhen-Hong Kong-Guangzhou overtaking Tokyo-Yokohama to claim the top spot and San Jose-San Francisco moving up three places to third position.
-   </description>
-   <pubDate>
-    Mon, 01 Sep 2025 09:00:00 +0200
-   </pubDate>
-  </item>
-  <item>
-   <title>
     WIPO Global Innovation Index 2025: Switzerland, Sweden, US, the Republic of Korea and Singapore Top Ranking; China Enters Top 10; Innovation Investment Growth Slows
    </title>
    <author>
@@ -211,6 +191,26 @@    </description>
    <pubDate>
     Fri, 13 Mar 2026 09:00:00 +0100
+   </pubDate>
+  </item>
+  <item>
+   <title>
+    WIPO Launches Artificial Intelligence Infrastructure Interchange
+   </title>
+   <author>
+    WIPO
+   </author>
+   <link>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0005.html
+   </link>
+   <guid>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0005.html
+   </guid>
+   <description>
+    WIPO today launched the Artificial Intelligence Infrastructure Interchange (AIII), a new initiative for expert dialogue on IP and AI issues that will focus on technical and operational issues and complement ongoing discussions at other WIPO forums.
+   </description>
+   <pubDate>
+    Tue, 17 Mar 2026 10:56:00 +0100
    </pubDate>
   </item>
  </channel>
