--- content_archive/ABC_News.xml	2026-02-13 06:57:32.859833260 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-13 06:58:15.428450157 +0000
@@ -32,101 +32,48 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    Bougainville mine that sparked a civil war on rapid path to reopening
+    Gaza man finds bones he says are unborn daughter killed in strike
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-12/bougainville-signs-secretive-deal-to-reopen-panguna-mine/106331778
+    https://www.abc.net.au/news/2026-02-12/gaza-father-searches-for-family-bones-in-house-rubble/106335798
    </link>
    <description>
-    The Autonomous Bougainville Government signs a secretive deal to reopen the huge Panguna gold and copper mine that sparked a catastrophic civil war three decades ago.
+    Mahmoud Hammad believes he has found his family's remains in the rubble of his house, which was destroyed more than two years ago in air strikes, killing his pregnant wife and their five children.
    </description>
-   <dc:creator>
-    Marian Faa and Theckla Gunga
-   </dc:creator>
    <pubDate>
-    Wed, 11 Feb 2026 18:52:41 +0000
+    Thu, 12 Feb 2026 08:43:59 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-12/bougainville-signs-secretive-deal-to-reopen-panguna-mine/106331778
+    https://www.abc.net.au/news/2026-02-12/gaza-father-searches-for-family-bones-in-house-rubble/106335798
    </guid>
    <category>
-    Mining and Metals Industry
-   </category>
-   <category>
-    Gold
-   </category>
-   <category>
-    Civil Unrest
-   </category>
-   <category>
-    Political Leadership
-   </category>
-   <category>
-    Copper
+    Unrest, Conflict and War
    </category>
    <media:group>
     <media:description>
-     A secretive deal has been signed to fast-track the reopening of the Panguna copper and gold mine.
+     Mahmoud Hammad believes the tiny remains belong to the unborn daughter his pregnant wife was carrying when an Israeli air strike destroyed their home more than two years ago, killing her and their five children.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=3991&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=2245&amp;xPos=873&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=3368&amp;xPos=312&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=1684&amp;xPos=1154&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=2993&amp;xPos=499&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=1262&amp;xPos=1365&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/32234a42238bb36d8eddc2fae37114c2?impolicy=wcms_crop_resize&amp;cropH=2245&amp;cropW=1684&amp;xPos=1154&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-   </media:group>
-  </item>
-  <item>
-   <title>
-    Former PM's son leads party to victory in post-uprising Bangladeshi election
-   </title>
-   <link>
-    https://www.abc.net.au/news/2026-02-13/bangladesh-nationalist-party-wins-first-election-since-uprising/106338732
-   </link>
-   <description>
-    The Bangladesh Nationalist Party has won a landmark parliamentary election, according to local TV stations, securing a resounding mandate in a pivotal vote.
-   </description>
-   <dc:creator>
-    Allyson Horn
-   </dc:creator>
-   <pubDate>
-    Fri, 13 Feb 2026 01:41:45 +0000
-   </pubDate>
-   <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-13/bangladesh-nationalist-party-wins-first-election-since-uprising/106338732
-   </guid>
-   <category>
-    National Elections
-   </category>
-   <category>
-    World Politics
-   </category>
-   <media:group>
-    <media:description>
-     Tarique Rahman will become the country's prime minister.
-    </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=261&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=3334&amp;cropW=3334&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=3334&amp;cropW=2501&amp;xPos=400&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=3334&amp;cropW=4445&amp;xPos=4&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=3334&amp;cropW=1875&amp;xPos=853&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/b24ba06395630e77be2fbd47ca7a713a?impolicy=wcms_crop_resize&amp;cropH=3334&amp;cropW=2501&amp;xPos=400&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1920&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1080&amp;xPos=420&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1620&amp;xPos=150&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1440&amp;xPos=240&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=608&amp;xPos=656&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/61ceb7dd2f3fbcb887c22f7cd6094355?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
     ICE to withdraw from Minnesota, Trump's border tsar announces
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-13/ice-withdrawal-minnesota-homan-trump-tsar/106339228
    </link>
    <description>
@@ -200,12 +147,44 @@
     </media:description>
     <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1458&amp;cropW=2592&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1728&amp;cropW=1728&amp;xPos=59&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1728&amp;cropW=2592&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1728&amp;cropW=1296&amp;xPos=275&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1728&amp;cropW=2304&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1728&amp;cropW=972&amp;xPos=437&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/99d610daa7ca4111115a393ac4cef055?impolicy=wcms_crop_resize&amp;cropH=1728&amp;cropW=1296&amp;xPos=275&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
+  <item>
+   <title>
+    Judge sets trial date for Donald Trump's defamation case against BBC
+   </title>
+   <link>
+    https://www.abc.net.au/news/2026-02-13/trial-date-set-donald-trump-defamation-case-bbc/106339012
+   </link>
+   <description>
+    A US District Judge has also rejected the BBC's application to halt the discovery phase of the $US10 billion ($15.1 billion) lawsuit.
+   </description>
+   <pubDate>
+    Thu, 12 Feb 2026 14:44:05 +0000
+   </pubDate>
+   <guid isPermaLink="true">
+    https://www.abc.net.au/news/2026-02-13/trial-date-set-donald-trump-defamation-case-bbc/106339012
+   </guid>
+   <category>
+    Media Industry
+   </category>
+   <media:group>
+    <media:description>
+     Donald Trump filed a $US10 billion lawsuit against the BBC over a documentary edit of him.
+    </media:description>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2078&amp;cropW=3695&amp;xPos=0&amp;yPos=196&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2471&amp;cropW=2471&amp;xPos=612&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2463&amp;cropW=3695&amp;xPos=0&amp;yPos=4&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2471&amp;cropW=1853&amp;xPos=921&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2471&amp;cropW=3295&amp;xPos=200&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2471&amp;cropW=1390&amp;xPos=1153&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/ddbe7a1ad7302a09a072db6f997ce8c6?impolicy=wcms_crop_resize&amp;cropH=2471&amp;cropW=1853&amp;xPos=921&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+   </media:group>
+  </item>
  </channel>
 </rss>
