--- content_archive/ABC_News.xml	2026-02-10 07:01:17.767509794 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-10 07:01:58.199241038 +0000
@@ -32,54 +32,60 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    The 'insane' reality of raising a nine-year-old in Trump and ICE's Minnesota
+    Blood clot revealed as cause of Catherine O'Hara's death
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-09/inside-minnesota-familes-at-risk-from-ice-arrest-speak-out/106322022
+    https://www.abc.net.au/news/2026-02-10/catherine-ohara-cause-of-death-announced/106324996
    </link>
    <description>
-    What was once a hindrance is now "a paranoia like I've never known before", one Minnesotan man says, as his family hides from immigration agents.
+    Award-winning actress Catherine O'Hara died from a pulmonary embolism, according to her death certificate.
    </description>
-   <dc:creator>
-    Rachel Clayton
-   </dc:creator>
    <pubDate>
-    Mon, 09 Feb 2026 06:16:43 +0000
+    Mon, 09 Feb 2026 21:58:51 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-09/inside-minnesota-familes-at-risk-from-ice-arrest-speak-out/106322022
+    https://www.abc.net.au/news/2026-02-10/catherine-ohara-cause-of-death-announced/106324996
    </guid>
    <category>
-    Civil Unrest
+    Celebrities
    </category>
    <category>
-    Immigration
+    Actor
+   </category>
+   <category>
+    Movies
+   </category>
+   <category>
+    Television
+   </category>
+   <category>
+    Arts, Culture and Entertainment
    </category>
    <media:group>
     <media:description>
-     The family spoke to the ABC on the condition of anonymity to protect them from being targeted by ICE.
+     Versatile comic actor Catherine O'Hara died in late January aged 71.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2385&amp;cropW=4240&amp;xPos=0&amp;yPos=224&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=2832&amp;xPos=704&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2826&amp;cropW=4240&amp;xPos=0&amp;yPos=3&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=2124&amp;xPos=1058&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=3776&amp;xPos=232&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=1593&amp;xPos=1324&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/549f620c4341737641d3bd720bad4cbf?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=2124&amp;xPos=1058&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=1969&amp;cropW=3500&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=2535&amp;cropW=2535&amp;xPos=399&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=3500&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=2535&amp;cropW=1901&amp;xPos=715&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=2535&amp;cropW=3380&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=2535&amp;cropW=1426&amp;xPos=953&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/95ee0734b9514386daa94c8da0b0fb0a?impolicy=wcms_crop_resize&amp;cropH=2535&amp;cropW=1901&amp;xPos=715&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
     King Charles will cooperate with police over any Andrew investigation, palace says
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-10/king-stands-ready-to-help-police-with-andrew-mountbatten-probe/106324734
    </link>
    <description>
