--- content_archive/WIPO_Press.xml+++ new_content@@ -15,26 +15,6 @@   </language>
   <item>
    <title>
-    WIPO Announces 2025 Global Awards Winners: Recognizing IP-Driven Innovation and Impact
-   </title>
-   <author>
-    WIPO
-   </author>
-   <link>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0007.html
-   </link>
-   <guid>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0007.html
-   </guid>
-   <description>
-    Ten SMEs and startups are the winners of this year’s WIPO Global Awards, with products including antibody therapeutics, sign-to-speech gloves, next-generation memory chips and sustainable protein fermentation—breakthroughs that reflect the transformative power of intellectual property (IP) in improving lives, advancing technology, and building more sustainable communities.
-   </description>
-   <pubDate>
-    Fri, 11 Jul 2025 12:00:00 +0200
-   </pubDate>
-  </item>
-  <item>
-   <title>
     Venture Capital Metric Reshapes WIPO's World’s Top 100 Innovation Clusters Rankings
    </title>
    <author>
@@ -211,6 +191,26 @@    </description>
    <pubDate>
     Fri, 06 Mar 2026 09:30:00 +0100
+   </pubDate>
+  </item>
+  <item>
+   <title>
+    Albania is First to Join WIPO’s Riyadh Design Law Treaty
+   </title>
+   <author>
+    WIPO
+   </author>
+   <link>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0004.html
+   </link>
+   <guid>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0004.html
+   </guid>
+   <description>
+    Albania today became the first member state to join WIPO’s Riyadh Design Law Treaty (RDLT), depositing its instrument of ratification with Director General Daren Tang and marking a major milestone in bringing into force the new international framework for design protection.
+   </description>
+   <pubDate>
+    Fri, 13 Mar 2026 09:00:00 +0100
    </pubDate>
   </item>
  </channel>
