--- content_archive/ABC_News.xml	2026-02-17 06:27:39.243501587 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-17 06:28:20.532784253 +0000
@@ -111,95 +111,101 @@
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/dbb85e394d29119ee122098b1bf5df99?impolicy=wcms_crop_resize&amp;cropH=683&amp;cropW=683&amp;xPos=171&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/dbb85e394d29119ee122098b1bf5df99?impolicy=wcms_crop_resize&amp;cropH=683&amp;cropW=1024&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/dbb85e394d29119ee122098b1bf5df99?impolicy=wcms_crop_resize&amp;cropH=683&amp;cropW=512&amp;xPos=256&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/dbb85e394d29119ee122098b1bf5df99?impolicy=wcms_crop_resize&amp;cropH=683&amp;cropW=911&amp;xPos=57&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/dbb85e394d29119ee122098b1bf5df99?impolicy=wcms_crop_resize&amp;cropH=683&amp;cropW=384&amp;xPos=320&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/dbb85e394d29119ee122098b1bf5df99?impolicy=wcms_crop_resize&amp;cropH=683&amp;cropW=512&amp;xPos=256&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Quiz: Think you know the Year of the Horse? Take the 10-question challenge
+    'Forced confessions' amid government crackdown spells execution fears
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-17/lunar-new-year-of-the-horse-quiz-china-south-korea-vietnam-asia/106348886
+    https://www.abc.net.au/news/2026-02-17/iran-forced-confessions-bahai-scapegoats-face-execution/106349538
    </link>
    <description>
-    How much do you know about the Year of the Horse and unique Lunar New Year traditions in Asian countries? Put your knowledge to the test with these 10 questions and see if you're a true Lunar New Year expert!
+    Families of Baha'i youth arrested by Iran's security forces say they've been subject to "forced confessions" and fear they may face the death penalty.
    </description>
    <dc:creator>
-    Xiaoning Mo
+    Nassim Khadem
    </dc:creator>
    <pubDate>
-    Mon, 16 Feb 2026 19:13:23 +0000
+    Mon, 16 Feb 2026 18:57:54 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-17/lunar-new-year-of-the-horse-quiz-china-south-korea-vietnam-asia/106348886
+    https://www.abc.net.au/news/2026-02-17/iran-forced-confessions-bahai-scapegoats-face-execution/106349538
    </guid>
    <category>
-    Community and Multicultural Festivals
+    World Politics
    </category>
    <category>
-    Human Interest
+    Civil Unrest
    </category>
    <category>
-    History
+    War
+   </category>
+   <category>
+    Religious Discrimination
+   </category>
+   <category>
+    Discrimination
+   </category>
+   <category>
+    War Crimes
    </category>
    <media:group>
     <media:description>
-     Draco Malfoy, whose Chinese name "Ma-er-fu" contains "Ma (Horse)" and "Fu (fortune)", became a festive mascot for the 2026 Year of the Horse, appearing on red decorations, phone covers, and merchandise, symbolizing luck and prosperity.
+     Venus Hosseini-Nejad's family fears for her safety.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=254&amp;cropW=451&amp;xPos=0&amp;yPos=172&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=451&amp;cropW=451&amp;xPos=0&amp;yPos=38&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=301&amp;cropW=451&amp;xPos=0&amp;yPos=149&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=598&amp;cropW=448&amp;xPos=1&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=338&amp;cropW=451&amp;xPos=0&amp;yPos=130&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=598&amp;cropW=336&amp;xPos=57&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=598&amp;cropW=448&amp;xPos=1&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=625&amp;cropW=1112&amp;xPos=0&amp;yPos=487&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=1112&amp;cropW=1112&amp;xPos=0&amp;yPos=244&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=741&amp;cropW=1112&amp;xPos=0&amp;yPos=429&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=1483&amp;cropW=1112&amp;xPos=0&amp;yPos=116&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=834&amp;cropW=1112&amp;xPos=0&amp;yPos=383&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=1600&amp;cropW=900&amp;xPos=209&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/cb512bd0cc5014556541e4e9c47f3db5?impolicy=wcms_crop_resize&amp;cropH=1483&amp;cropW=1112&amp;xPos=0&amp;yPos=116&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Ukrainian ex-minister accused of stashing laundered millions offshore
+    Quiz: Think you know the Year of the Horse? Take the 10-question challenge
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-17/ukraine-herman-halushchenko-corruption-allegations/106352078
+    https://www.abc.net.au/news/2026-02-17/lunar-new-year-of-the-horse-quiz-china-south-korea-vietnam-asia/106348886
    </link>
    <description>
-    Ukraine's anti-corruption police have accused its ex-energy minister of helping launder kickbacks and stashing millions offshore, a day after he was detained trying to leave the country.
+    How much do you know about the Year of the Horse and unique Lunar New Year traditions in Asian countries? Put your knowledge to the test with these 10 questions and see if you're a true Lunar New Year expert!
    </description>
+   <dc:creator>
+    Xiaoning Mo
+   </dc:creator>
    <pubDate>
-    Mon, 16 Feb 2026 17:31:47 +0000
+    Mon, 16 Feb 2026 19:13:23 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-17/ukraine-herman-halushchenko-corruption-allegations/106352078
+    https://www.abc.net.au/news/2026-02-17/lunar-new-year-of-the-horse-quiz-china-south-korea-vietnam-asia/106348886
    </guid>
    <category>
-    World Politics
-   </category>
-   <category>
-    Corruption
-   </category>
-   <category>
-    Bribery
+    Community and Multicultural Festivals
    </category>
    <category>
-    Business, Economics and Finance
+    Human Interest
    </category>
    <category>
-    Law, Crime and Justice
+    History
    </category>
    <media:group>
     <media:description>
-     Herman Halushchenko served briefly as justice minister before resigning over the corruption case.
+     Draco Malfoy, whose Chinese name "Ma-er-fu" contains "Ma (Horse)" and "Fu (fortune)", became a festive mascot for the 2026 Year of the Horse, appearing on red decorations, phone covers, and merchandise, symbolizing luck and prosperity.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2089&amp;cropW=3713&amp;xPos=0&amp;yPos=90&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2653&amp;cropW=2653&amp;xPos=559&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2475&amp;cropW=3713&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2653&amp;cropW=1990&amp;xPos=891&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2653&amp;cropW=3537&amp;xPos=117&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2653&amp;cropW=1492&amp;xPos=1140&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/3c706e60e22e30a53b77ba1afa7ad8f5?impolicy=wcms_crop_resize&amp;cropH=2653&amp;cropW=1990&amp;xPos=891&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=254&amp;cropW=451&amp;xPos=0&amp;yPos=172&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=451&amp;cropW=451&amp;xPos=0&amp;yPos=38&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=301&amp;cropW=451&amp;xPos=0&amp;yPos=149&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=598&amp;cropW=448&amp;xPos=1&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=338&amp;cropW=451&amp;xPos=0&amp;yPos=130&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=598&amp;cropW=336&amp;xPos=57&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/39ae2eb90809be59548039ba13d55121?impolicy=wcms_crop_resize&amp;cropH=598&amp;cropW=448&amp;xPos=1&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
