--- content_archive/ABC_News.xml	2026-02-08 12:44:35.341115266 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-08 12:44:56.430083511 +0000
@@ -32,55 +32,20 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    Americans propose an end to Russian-Ukrainian war by June, Zelenskyy says
-   </title>
-   <link>
-    https://www.abc.net.au/news/2026-02-07/russia-ukraine-deadline-to-end-war-/106317976
-   </link>
-   <description>
-    America has given Russia and Ukraine a June deadline to end the war — and will put pressure on both sides to ensure it is met — Volodymyr Zelenskyy says.
-   </description>
-   <pubDate>
-    Sat, 07 Feb 2026 12:17:54 +0000
-   </pubDate>
-   <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-07/russia-ukraine-deadline-to-end-war-/106317976
-   </guid>
-   <category>
-    Unrest, Conflict and War
-   </category>
-   <category>
-    World Politics
-   </category>
-   <media:group>
-    <media:description>
-     Volodymyr Zelenskyy says the US has given Ukraine and Russia a June deadline to end the war.
-    </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=260&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=3333&amp;xPos=834&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=4444&amp;xPos=278&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=1875&amp;xPos=1563&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/d0376fbb54d7e02b4e864ea7e2478d9c?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-   </media:group>
-  </item>
-  <item>
-   <title>
     Bangladesh's long road from uprising to accountability as it heads to the polls
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-08/bangladesh-returns-to-the-ballot-with-its-wounds-still-open/106296274
    </link>
    <description>
     Bangladesh is heading back to the ballot box in what is being billed as its first free and fair election in more than a decade.
    </description>
    <dc:creator>
     Meghna Bali and Bhat Burhan
@@ -105,20 +70,52 @@
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/264764b2ac8f258db0a040eb34491208?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=2832&amp;xPos=704&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/264764b2ac8f258db0a040eb34491208?impolicy=wcms_crop_resize&amp;cropH=2827&amp;cropW=4240&amp;xPos=0&amp;yPos=3&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/264764b2ac8f258db0a040eb34491208?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=2124&amp;xPos=1058&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/264764b2ac8f258db0a040eb34491208?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=3776&amp;xPos=232&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/264764b2ac8f258db0a040eb34491208?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=1593&amp;xPos=1324&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/264764b2ac8f258db0a040eb34491208?impolicy=wcms_crop_resize&amp;cropH=2832&amp;cropW=2124&amp;xPos=1058&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
+    Korean cryptocurrency exchange accidentally gives away $60b in bitcoin
+   </title>
+   <link>
+    https://www.abc.net.au/news/2026-02-08/korean-exchange-bithumb-accidentally-gives-away-60b-bitcoin/106318826
+   </link>
+   <description>
+    A South Korean cryptocurrency exchange apologises after mistakenly transferring more than $60 billion worth of bitcoin to users, which briefly prompted a sell-off on the platform.
+   </description>
+   <pubDate>
+    Sat, 07 Feb 2026 23:24:21 +0000
+   </pubDate>
+   <guid isPermaLink="true">
+    https://www.abc.net.au/news/2026-02-08/korean-exchange-bithumb-accidentally-gives-away-60b-bitcoin/106318826
+   </guid>
+   <category>
+    Cryptocurrency
+   </category>
+   <media:group>
+    <media:description>
+     The error caused temporary volatility in bitcoin prices on the platform.
+    </media:description>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=1969&amp;cropW=3500&amp;xPos=0&amp;yPos=182&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=2333&amp;xPos=584&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=3499&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=1750&amp;xPos=875&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=3111&amp;xPos=195&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=1312&amp;xPos=1094&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/ff595f5a89bf35a824ad7cc60011f2d8?impolicy=wcms_crop_resize&amp;cropH=2333&amp;cropW=1750&amp;xPos=875&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+   </media:group>
+  </item>
+  <item>
+   <title>
     WHO says one person dead from Nipah virus in Bangladesh
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-08/who-says-one-person-dead-from-nipah-virus-in-bangladesh/106318702
    </link>
    <description>
     A woman dies in northern Bangladesh after contracting the deadly Nipah virus infection.
    </description>
    <pubDate>
     Sat, 07 Feb 2026 22:24:50 +0000
