--- content_archive/ABC_News.xml	2026-02-06 06:52:27.416318182 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-06 06:52:57.493282178 +0000
@@ -32,168 +32,156 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    Australians in Japan shaken by ski deaths but say its snowfields are safe
+    Minneapolis man charged with threatening, cyberstalking ICE officers
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-05/australians-in-japan-say-ski-fields-are-safe/106307558
+    https://www.abc.net.au/news/2026-02-06/minneapolis-man-charged-cyberstalking-ice/106313730
    </link>
    <description>
-    Australians working in Japan's ski fields say while the sport has unpredictable risks, there are ways to be prepared.
+    A 37-year-old activist has been arrested for what the US Justice Department says is cyberstalking and threatening to kill or assault ICE officers involved in the crackdown in Minnesota.
    </description>
-   <dc:creator>
-    Allyson Horn
-   </dc:creator>
    <pubDate>
-    Thu, 05 Feb 2026 01:26:32 +0000
+    Fri, 06 Feb 2026 03:34:02 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-05/australians-in-japan-say-ski-fields-are-safe/106307558
+    https://www.abc.net.au/news/2026-02-06/minneapolis-man-charged-cyberstalking-ice/106313730
    </guid>
    <category>
-    Disasters, Accidents and Emergency Incidents
+    World Politics
+   </category>
+   <category>
+    Crime
    </category>
    <media:group>
     <media:description>
-     Japan's snowfields are a major tourism destination for Australians each year.
+     White House press secretary Karoline Leavitt holds a photo of Kyle Wagner.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=361&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=3268&amp;cropW=3268&amp;xPos=918&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=3268&amp;cropW=4902&amp;xPos=98&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=3268&amp;cropW=2451&amp;xPos=990&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=3268&amp;cropW=4357&amp;xPos=373&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=3268&amp;cropW=1838&amp;xPos=2052&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/1522d86efb29b72f5d7d5b655852a4f6?impolicy=wcms_crop_resize&amp;cropH=3268&amp;cropW=2451&amp;xPos=990&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2158&amp;cropW=3836&amp;xPos=0&amp;yPos=200&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2557&amp;cropW=2557&amp;xPos=640&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2557&amp;cropW=3835&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2557&amp;cropW=1918&amp;xPos=959&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2557&amp;cropW=3409&amp;xPos=213&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2557&amp;cropW=1438&amp;xPos=1199&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/71b52bced382b1daeb2c4cf95f3cf4ab?impolicy=wcms_crop_resize&amp;cropH=2557&amp;cropW=1918&amp;xPos=959&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Starmer says sorry to Epstein's victims for appointing Mandelson as US ambassador
+    Dozens of skiers rescued at New York mountain
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-05/starmer-apologises-epstein-vicitims-lord-mandelson/106311694
+    https://www.abc.net.au/news/2026-02-06/nearly-70-people-rescued-from-new-york-ski-mountain-gondolas/106312400
    </link>
    <description>
-    British Prime Minister Keir Starmer says former ambassador to the US Peter Mandelson lied when asked about the nature of his relationship with convicted sex offender Jeffrey Epstein.
+    Rescuers have saved nearly 70 skiers who were stranded for hours, with some trapped up to 21 metres in the air, after a gondola malfunction in New York.
    </description>
-   <dc:creator>
-    Rudi Maxwell
-   </dc:creator>
    <pubDate>
-    Thu, 05 Feb 2026 12:51:10 +0000
+    Thu, 05 Feb 2026 22:17:27 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-05/starmer-apologises-epstein-vicitims-lord-mandelson/106311694
+    https://www.abc.net.au/news/2026-02-06/nearly-70-people-rescued-from-new-york-ski-mountain-gondolas/106312400
    </guid>
    <category>
-    World Politics
-   </category>
-   <category>
-    Sexual Offences
+    Disasters, Accidents and Emergency Incidents
    </category>
    <media:group>
     <media:description>
-     Keir Starmer has apologised to victims of convicted sex offender Jeffrey Epstein.
+     Between 60 and 70 skiers had to be rescued from a malfunctioning gondola on Gore Mountain in upstate New York.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=147&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=3333&amp;xPos=629&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1045&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=4444&amp;xPos=504&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=1875&amp;xPos=1358&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/f2b1c3430c3685074f3794f6bab4a78e?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1045&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1920&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1080&amp;xPos=420&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1620&amp;xPos=150&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1440&amp;xPos=240&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=608&amp;xPos=656&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/f86dfc571ade4e7f4cabd481efee35f4?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Russian captain of cargo ship jailed over fatal North Sea collision
+    US wants world to follow Australia into 'club of nations' to counter China
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-06/north-sea-collision-captain-jailed/106311942
+    https://www.abc.net.au/news/2026-02-06/us-pitches-new-critical-minerals-trading-bloc/106309202
    </link>
    <description>
-    A judge describes Vladimir Motin as "an accident waiting to happen" as he hands the Russian national a six-year jail term.
+    The US is urging dozens of countries to follow Australia's lead and sign on to a plan to stop relying on China for the critical minerals used in modern technology.
    </description>
+   <dc:creator>
+    Brad Ryan
+   </dc:creator>
    <pubDate>
-    Thu, 05 Feb 2026 15:55:48 +0000
+    Thu, 05 Feb 2026 19:22:12 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-06/north-sea-collision-captain-jailed/106311942
+    https://www.abc.net.au/news/2026-02-06/us-pitches-new-critical-minerals-trading-bloc/106309202
    </guid>
    <category>
-    Courts
+    World Politics
    </category>
    <category>
-    Disasters, Accidents and Emergency Incidents
+    Trade Agreements
    </category>
    <category>
-    Marine Incidents
+    Mining and Metals Industry
    </category>
    <category>
-    Crime
-   </category>
-   <category>
-    Navy
+    Foreign Affairs
    </category>
    <media:group>
     <media:description>
-     Russian national Vladimir Motin, captain of the Portuguese-flagged Solong, has been jailed for six years over the collision with the Stena Immaculate tanker.
+     JD Vance did not mention China by name, but said "foreign supply" often flooded markets to make new critical minerals projects unviable.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=417&amp;cropW=741&amp;xPos=0&amp;yPos=159&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=741&amp;cropW=741&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=494&amp;cropW=741&amp;xPos=0&amp;yPos=121&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=873&amp;cropW=655&amp;xPos=51&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=556&amp;cropW=741&amp;xPos=0&amp;yPos=90&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=873&amp;cropW=491&amp;xPos=133&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/f63e30ab67998fb1676b1f59749e9563?impolicy=wcms_crop_resize&amp;cropH=873&amp;cropW=655&amp;xPos=51&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2030&amp;cropW=3609&amp;xPos=0&amp;yPos=188&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=2406&amp;xPos=473&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=3609&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=1805&amp;xPos=759&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=3208&amp;xPos=201&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=1353&amp;xPos=1055&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=1805&amp;xPos=759&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Russia and Ukraine conduct POW swap after latest US-led peace talks
+    End of US-Russia nuclear weapons pact 'could not come at a worse time'
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-06/ukraine-russia-talks-united-states-trump/106312078
+    https://www.abc.net.au/news/2026-02-06/us-russia-nuclear-weapons-treaty-expires/106312346
    </link>
    <description>
-    The Kremlin says it sees progress and positive movement in the talks, brokered by the United States, as Donald Trump urges both sides to reach a peace deal.
+    The last remaining nuclear arms pact between Russia and the United States has expired after Donald Trump rejected an offer from his Russian counterpart.
    </description>
    <pubDate>
-    Thu, 05 Feb 2026 17:01:52 +0000
+    Thu, 05 Feb 2026 22:35:33 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-06/ukraine-russia-talks-united-states-trump/106312078
+    https://www.abc.net.au/news/2026-02-06/us-russia-nuclear-weapons-treaty-expires/106312346
    </guid>
    <category>
-    Unrest, Conflict and War
-   </category>
-   <category>
-    World Politics
+    Nuclear Weapons
    </category>
    <category>
-    Territorial Disputes
-   </category>
-   <category>
-    Foreign Affairs
+    Unrest, Conflict and War
    </category>
    <media:group>
     <media:description>
-     Video released by the Russian Defence Ministry shows what it said to be Russian service personnel captured by Ukrainian forces after being released.
+     The pact's termination leaves no caps on the two largest atomic arsenals for the first time in half a century.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=2560&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=1440&amp;xPos=560&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=2160&amp;xPos=200&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=1080&amp;xPos=740&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=1920&amp;xPos=320&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=810&amp;xPos=875&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/e991a8f7f84f233709e5bc1b9e34184d?impolicy=wcms_crop_resize&amp;cropH=1440&amp;cropW=1080&amp;xPos=740&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=170&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=3152&amp;xPos=924&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=4728&amp;xPos=136&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=2364&amp;xPos=1318&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=4203&amp;xPos=399&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=1773&amp;xPos=1614&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=2364&amp;xPos=1318&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
