--- content_archive/IPA_what-are-tradmarks.md+++ new_content@@ -7,9 +7,9 @@ * icon / 24 / shield
 
   Protects your brand, products or services
-* Costs start at $100,000
-* Takes at least 24 months to register
-* Lasts up to 2 years before renewal
+* Costs start at $250
+* Takes at least 7 months to register
+* Lasts up to 10 years before renewal
 
 ## 
 
@@ -27,7 +27,7 @@ 
 ## How do trade marks work?
 
-Here's a quick video that explains what trade marks are and how they can protect your brand, products or services. We've also added an image to aid in the explanation.
+Here's a quick video that explains what trade marks are and how they can protect your brand, products or services.
 
 Skip embedded contentReturn to page content
 
@@ -137,8 +137,7 @@ 
 * Notifying other traders that your trade mark is registered in Australia
 * Indicating you have strong rights that prevent others from using the same or similar trade mark
-* Deterring someone from infringing your IP
-* Preventing someone from stealing your first-born child.
+* Deterring someone from infringing your IP.
 
 **Do I have to use the symbol?** No, you don't have to use it. You'll maintain your exclusive rights either way.  
   
@@ -162,6 +161,4 @@ 
 Accept
 
-Opt Out
-
-Skip embedded contentReturn to page content
+Opt Out