--- content_archive/WIPO_Press.xml	2026-02-12 13:03:21.338373233 +0000
+++ content_archive/WIPO_Press.xml.tmp	2026-02-12 13:03:40.000501910 +0000
@@ -8,40 +8,20 @@
    https://www.wipo.int/pressroom/en/articles/
   </link>
   <description>
    WIPO Press Releases
   </description>
   <language>
    en
   </language>
   <item>
    <title>
-    World Intellectual Property Day 2025 Celebrates Musicians
-   </title>
-   <author>
-    WIPO
-   </author>
-   <link>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0004.html
-   </link>
-   <guid>
-    https://www.wipo.int/pressroom/en/articles/2025/article_0004.html
-   </guid>
-   <description>
-    Pop music continues to dominate global charts while reggaeton and K-pop are gaining fans among the ranks of the most-popular musical genres, according to new WIPO data released to mark World Intellectual Property (IP) Day, which is commemorated each April 26.
-   </description>
-   <pubDate>
-    Fri, 25 Apr 2025 09:30:00 +0200
-   </pubDate>
-  </item>
-  <item>
-   <title>
     Investment in Intangible Assets Surges, led by Funding for Software and Databases Amid AI Boom
    </title>
    <author>
     WIPO
    </author>
    <link>
     https://www.wipo.int/pressroom/en/articles/2025/article_0005.html
    </link>
    <guid>
     https://www.wipo.int/pressroom/en/articles/2025/article_0005.html
@@ -206,12 +186,32 @@
    <guid>
     https://www.wipo.int/pressroom/en/articles/2025/article_0013.html
    </guid>
    <description>
     Global awareness and trust in intellectual property (IP) systems has reached new heights globally, according to WIPO Pulse 2025, the world's most comprehensive survey of IP perception and awareness.
    </description>
    <pubDate>
     Thu, 13 Nov 2025 06:43:00 +0100
    </pubDate>
   </item>
+  <item>
+   <title>
+    WIPO Coordination Committee Nominates Singapore’s Daren Tang for Second Term as Director General
+   </title>
+   <author>
+    WIPO
+   </author>
+   <link>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0001.html
+   </link>
+   <guid>
+    https://www.wipo.int/pressroom/en/articles/2026/article_0001.html
+   </guid>
+   <description>
+    The Coordination Committee of the World Intellectual Property Organization (WIPO) has nominated Mr. Daren Tang from Singapore for a second six year term to the post of Director General.
+   </description>
+   <pubDate>
+    Thu, 12 Feb 2026 12:00:00 +0100
+   </pubDate>
+  </item>
  </channel>
 </rss>
