--- content_archive/ABC_News.xml	2026-02-15 06:49:38.022498792 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-15 06:50:41.718466844 +0000
@@ -32,64 +32,20 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    Australia and EU on the cusp of signing landmark free trade agreement
-   </title>
-   <link>
-    https://www.abc.net.au/news/2026-02-14/australia-and-eu-sign-landmark-free-trade-agreement/106344826
-   </link>
-   <description>
-    Australia and the European Union are on the cusp of finally reaching a free trade agreement after years of sometimes fraught negotiations.
-   </description>
-   <dc:creator>
-    Stephen Dziedzic
-   </dc:creator>
-   <pubDate>
-    Sat, 14 Feb 2026 00:28:39 +0000
-   </pubDate>
-   <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-14/australia-and-eu-sign-landmark-free-trade-agreement/106344826
-   </guid>
-   <category>
-    Trade
-   </category>
-   <category>
-    Federal Government
-   </category>
-   <category>
-    Foreign Affairs
-   </category>
-   <category>
-    Government and Politics
-   </category>
-   <media:group>
-    <media:description>
-     After days of discussion, Australia is one step closer to closing a deal with the EU.
-    </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=2812&amp;cropW=5000&amp;xPos=0&amp;yPos=261&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=3335&amp;xPos=832&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=1&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=2501&amp;xPos=2096&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=4447&amp;xPos=277&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=1876&amp;xPos=1562&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c3afc1803c17de23d9d2182bae332a12?impolicy=wcms_crop_resize&amp;cropH=3335&amp;cropW=2501&amp;xPos=2096&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-   </media:group>
-  </item>
-  <item>
-   <title>
     How Indonesians are fighting plummeting marriage rates
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-14/indonesian-marriage-rate-plummets-despite-demand/106334832
    </link>
    <description>
     Marriage is still a life goal in Indonesia, but the number of marriages has plummeted over the past decade.
    </description>
    <dc:creator>
     Natasya Salim
@@ -158,20 +114,52 @@
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=2400&amp;xPos=920&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=3600&amp;xPos=320&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1800&amp;xPos=1220&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=3200&amp;xPos=520&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1350&amp;xPos=1445&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1800&amp;xPos=1220&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
+    Police operation underway related to Nancy Guthrie
+   </title>
+   <link>
+    https://www.abc.net.au/news/2026-02-14/police-operation-kidnap-nancy-savannah-guthrie-in-tucson-arizona/106345266
+   </link>
+   <description>
+    Law enforcement in Arizona is conducting activity related to the Nancy Guthrie case.
+   </description>
+   <pubDate>
+    Sat, 14 Feb 2026 07:22:42 +0000
+   </pubDate>
+   <guid isPermaLink="true">
+    https://www.abc.net.au/news/2026-02-14/police-operation-kidnap-nancy-savannah-guthrie-in-tucson-arizona/106345266
+   </guid>
+   <category>
+    Crime
+   </category>
+   <media:group>
+    <media:description>
+     Savannah Guthrie has not been on the Today Show since the disappearance of her mother Nancy.
+    </media:description>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=608&amp;cropW=1080&amp;xPos=0&amp;yPos=177&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1025&amp;cropW=1025&amp;xPos=27&amp;yPos=14&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=720&amp;cropW=1080&amp;xPos=0&amp;yPos=102&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=133&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=810&amp;cropW=1080&amp;xPos=0&amp;yPos=65&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1077&amp;cropW=606&amp;xPos=232&amp;yPos=3&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=133&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+   </media:group>
+  </item>
+  <item>
+   <title>
     Russia poisoned Alexei Navalny, UK and allies say
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-15/russia-poisoned-alexei-navalny-uk-allies-say/106345766
    </link>
    <description>
     Five European foreign ministers say analysis of samples from the former Russian opposition leader, who died two years ago, confirmed the presence of a toxin from poison frogs.
    </description>
    <pubDate>
     Sat, 14 Feb 2026 13:24:14 +0000
