--- content_archive/ABC_News.xml	2026-02-06 18:46:46.157631577 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-06 18:47:10.259590094 +0000
@@ -158,54 +158,45 @@
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c54beeb85239ac050e329535622c1b1e?impolicy=wcms_crop_resize&amp;cropH=2250&amp;cropW=2250&amp;xPos=375&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c54beeb85239ac050e329535622c1b1e?impolicy=wcms_crop_resize&amp;cropH=2000&amp;cropW=3000&amp;xPos=0&amp;yPos=125&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c54beeb85239ac050e329535622c1b1e?impolicy=wcms_crop_resize&amp;cropH=2250&amp;cropW=1688&amp;xPos=858&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c54beeb85239ac050e329535622c1b1e?impolicy=wcms_crop_resize&amp;cropH=2250&amp;cropW=3000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c54beeb85239ac050e329535622c1b1e?impolicy=wcms_crop_resize&amp;cropH=2250&amp;cropW=1266&amp;xPos=1199&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c54beeb85239ac050e329535622c1b1e?impolicy=wcms_crop_resize&amp;cropH=2250&amp;cropW=1688&amp;xPos=858&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    US wants world to follow Australia into 'club of nations' to counter China
+    End of US-Russia nuclear weapons pact 'could not come at a worse time'
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-06/us-pitches-new-critical-minerals-trading-bloc/106309202
+    https://www.abc.net.au/news/2026-02-06/us-russia-nuclear-weapons-treaty-expires/106312346
    </link>
    <description>
-    The US is urging dozens of countries to follow Australia's lead and sign on to a plan to stop relying on China for the critical minerals used in modern technology.
+    The last remaining nuclear arms pact between Russia and the United States has expired after Donald Trump rejected an offer from his Russian counterpart.
    </description>
-   <dc:creator>
-    Brad Ryan
-   </dc:creator>
    <pubDate>
-    Thu, 05 Feb 2026 19:22:12 +0000
+    Thu, 05 Feb 2026 22:35:33 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-06/us-pitches-new-critical-minerals-trading-bloc/106309202
+    https://www.abc.net.au/news/2026-02-06/us-russia-nuclear-weapons-treaty-expires/106312346
    </guid>
    <category>
-    World Politics
+    Nuclear Weapons
    </category>
    <category>
-    Trade Agreements
-   </category>
-   <category>
-    Mining and Metals Industry
-   </category>
-   <category>
-    Foreign Affairs
+    Unrest, Conflict and War
    </category>
    <media:group>
     <media:description>
-     JD Vance did not mention China by name, but said "foreign supply" often flooded markets to make new critical minerals projects unviable.
+     The pact's termination leaves no caps on the two largest atomic arsenals for the first time in half a century.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2030&amp;cropW=3609&amp;xPos=0&amp;yPos=188&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=2406&amp;xPos=473&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=3609&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=1805&amp;xPos=759&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=3208&amp;xPos=201&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=1353&amp;xPos=1055&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/5e9f320acfb4c212ebb15a2030cfed1d?impolicy=wcms_crop_resize&amp;cropH=2406&amp;cropW=1805&amp;xPos=759&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=170&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=3152&amp;xPos=924&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=4728&amp;xPos=136&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=2364&amp;xPos=1318&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=4203&amp;xPos=399&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=1773&amp;xPos=1614&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c69f5626f31ed1b7a79a9bd74b2cafec?impolicy=wcms_crop_resize&amp;cropH=3152&amp;cropW=2364&amp;xPos=1318&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
