--- content_archive/ABC_News.xml	2026-02-16 01:59:49.700221778 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-16 02:00:36.938154311 +0000
@@ -32,156 +32,153 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    How Indonesians are fighting plummeting marriage rates
+    Six mysterious 'Twin Peaks' deaths in Balkan Mountains stir controversy
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-14/indonesian-marriage-rate-plummets-despite-demand/106334832
+    https://www.abc.net.au/news/2026-02-15/bulgaria-twin-peaks-balkan-mountains-murders/106336984
    </link>
    <description>
-    Marriage is still a life goal in Indonesia, but the number of marriages has plummeted over the past decade.
+    Six unsolved deaths in two remote parts of the Balkan Mountains become a source of conspiracy across Bulgaria.
    </description>
    <dc:creator>
-    Natasya Salim
+    Brianna Morris-Grant
    </dc:creator>
    <pubDate>
-    Fri, 13 Feb 2026 21:42:04 +0000
+    Sun, 15 Feb 2026 01:45:53 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-14/indonesian-marriage-rate-plummets-despite-demand/106334832
+    https://www.abc.net.au/news/2026-02-15/bulgaria-twin-peaks-balkan-mountains-murders/106336984
    </guid>
    <category>
-    Marriage
+    Homicide
    </category>
    <category>
-    Dating
+    Law, Crime and Justice
    </category>
    <media:group>
     <media:description>
-     Talitha Rarasty (left) works as a makeup artist in Surabaya, East Java, Indonesia.
+     CCTV footage showing six people found dead in west Bulgarian mountains hugging each other and bidding farewell on the day of the killings.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=416&amp;cropW=739&amp;xPos=0&amp;yPos=338&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=739&amp;cropW=739&amp;xPos=0&amp;yPos=167&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=493&amp;cropW=739&amp;xPos=0&amp;yPos=273&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=985&amp;cropW=739&amp;xPos=0&amp;yPos=302&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=554&amp;cropW=739&amp;xPos=0&amp;yPos=232&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=1297&amp;cropW=730&amp;xPos=5&amp;yPos=146&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/617f2f3ad4c4bc874f90eef140fa37a0?impolicy=wcms_crop_resize&amp;cropH=985&amp;cropW=739&amp;xPos=0&amp;yPos=302&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1920&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1080&amp;xPos=420&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1620&amp;xPos=150&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=491&amp;cropW=654&amp;xPos=643&amp;yPos=58&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=770&amp;cropW=433&amp;xPos=892&amp;yPos=65&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Australian craft brewers hop on 'clean and easy' Japanese lagers
+    Gisèle Pelicot intends to visit ex-husband in prison to get answers
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-14/japanese-lagers-thrive-in-australia-after-record-travel-numbers/106119108
+    https://www.abc.net.au/news/2026-02-15/gis%C3%A8le-pelicot-intends-to-visit-abuser-in-prison-to-get-answers/106346438
    </link>
    <description>
-    As record numbers of Australians continue to visit Japan, Japanese brewers have come to dominate our domestic beer market.
+    Gisèle Pelicot, the woman at the centre of France's largest rape trial, says she intends to visit her ex-husband in jail and look her abuser "straight in the eye" to get answers to other allegations under investigation.
    </description>
    <dc:creator>
-    Max Walden and Lachlan Bennett
+    Jessica Riga
    </dc:creator>
    <pubDate>
-    Fri, 13 Feb 2026 19:22:09 +0000
+    Sun, 15 Feb 2026 01:02:04 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-14/japanese-lagers-thrive-in-australia-after-record-travel-numbers/106119108
+    https://www.abc.net.au/news/2026-02-15/gis%C3%A8le-pelicot-intends-to-visit-abuser-in-prison-to-get-answers/106346438
    </guid>
    <category>
-    Travel and Tourism
+    Sexual Offences
    </category>
    <category>
-    Food and Beverage Processing Industry
-   </category>
-   <category>
-    Hospitality Industry
-   </category>
-   <category>
-    Drinks
+    Law, Crime and Justice
    </category>
    <media:group>
     <media:description>
-     Brewers attribute the crisp flavour of Japanese beer to the country's "soft" water.
+     Gisèle Pelicot has done multiple in-depth interviews ahead of the release of her memoir.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2385&amp;cropW=4240&amp;xPos=0&amp;yPos=7&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=2400&amp;xPos=920&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=3600&amp;xPos=320&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1800&amp;xPos=1220&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=3200&amp;xPos=520&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1350&amp;xPos=1445&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/2929778fbe5ee83aeb30c941e9b28ba0?impolicy=wcms_crop_resize&amp;cropH=2400&amp;cropW=1800&amp;xPos=1220&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=50&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=3333&amp;xPos=986&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1402&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=4444&amp;xPos=430&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=1875&amp;xPos=1715&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/ef87dc8e042dced18e3c7aa3d910352b?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1402&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Police operation underway related to Nancy Guthrie
+    New Zealand's North Island braces for more rain amid widespread flooding
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-14/police-operation-kidnap-nancy-savannah-guthrie-in-tucson-arizona/106345266
+    https://www.abc.net.au/news/2026-02-15/new-zealand-north-island-rain-flooding/106347264
    </link>
    <description>
-    Law enforcement in Arizona is conducting activity related to the Nancy Guthrie case.
+    Floods cause power outages, road collapses, home evacuations and are linked to the death of a man whose vehicle was submerged on a highway.
    </description>
    <pubDate>
-    Sat, 14 Feb 2026 07:22:42 +0000
+    Sun, 15 Feb 2026 10:19:27 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-14/police-operation-kidnap-nancy-savannah-guthrie-in-tucson-arizona/106345266
+    https://www.abc.net.au/news/2026-02-15/new-zealand-north-island-rain-flooding/106347264
    </guid>
    <category>
-    Crime
+    Floods
    </category>
    <media:group>
     <media:description>
-     Savannah Guthrie has not been on the Today Show since the disappearance of her mother Nancy.
+     Flooding in Ōtorohanga, New Zealand.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=608&amp;cropW=1080&amp;xPos=0&amp;yPos=177&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1025&amp;cropW=1025&amp;xPos=27&amp;yPos=14&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=720&amp;cropW=1080&amp;xPos=0&amp;yPos=102&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=133&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=810&amp;cropW=1080&amp;xPos=0&amp;yPos=65&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1077&amp;cropW=606&amp;xPos=232&amp;yPos=3&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/5dbbc5ae4d06f45513d446d834eb9ede?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=133&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=591&amp;cropW=1050&amp;xPos=0&amp;yPos=33&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=656&amp;cropW=656&amp;xPos=197&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=656&amp;cropW=984&amp;xPos=33&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=656&amp;cropW=492&amp;xPos=279&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=656&amp;cropW=875&amp;xPos=88&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=656&amp;cropW=369&amp;xPos=341&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/821b7d9a77b6ea3583ea1ceea3ea53b0?impolicy=wcms_crop_resize&amp;cropH=656&amp;cropW=492&amp;xPos=279&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    Russia poisoned Alexei Navalny, UK and allies say
+    Obama deplores lack of shame after Trump shares monkey clip
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-15/russia-poisoned-alexei-navalny-uk-allies-say/106345766
+    https://www.abc.net.au/news/2026-02-15/obama-trump-racist-monkey-clip-podcast/106346866
    </link>
    <description>
-    Five European foreign ministers say analysis of samples from the former Russian opposition leader, who died two years ago, confirmed the presence of a toxin from poison frogs.
+    Former US president Barack Obama has criticised a lack of shame and decorum in the country's political discourse, responding for the first time to a post on Donald Trump's social media account that depicted him and first lady Michelle as monkeys.
    </description>
    <pubDate>
-    Sat, 14 Feb 2026 13:24:14 +0000
+    Sun, 15 Feb 2026 03:53:45 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-15/russia-poisoned-alexei-navalny-uk-allies-say/106345766
+    https://www.abc.net.au/news/2026-02-15/obama-trump-racist-monkey-clip-podcast/106346866
    </guid>
    <category>
-    Crime
+    World Politics
+   </category>
+   <category>
+    Racism
    </category>
    <media:group>
     <media:description>
-     Alexei Navalny died in a Russian prison in February 2024.
+     Barack Obama and Donald Trump before the state funeral service for former US president Jimmy Carter in 2025.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=259&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=3330&amp;xPos=835&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=4995&amp;xPos=2&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=2498&amp;xPos=1251&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=4440&amp;xPos=280&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=1873&amp;xPos=1564&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/ea84d1b389b5620f730177c85218370c?impolicy=wcms_crop_resize&amp;cropH=3330&amp;cropW=2498&amp;xPos=1251&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=2444&amp;cropW=4345&amp;xPos=0&amp;yPos=236&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4345&amp;cropW=4345&amp;xPos=0&amp;yPos=157&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=2897&amp;cropW=4345&amp;xPos=0&amp;yPos=236&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4827&amp;cropW=3620&amp;xPos=252&amp;yPos=173&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=3259&amp;cropW=4345&amp;xPos=0&amp;yPos=189&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4953&amp;cropW=2786&amp;xPos=787&amp;yPos=47&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4827&amp;cropW=3620&amp;xPos=252&amp;yPos=173&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
