--- content_archive/ABC_News.xml	2026-02-16 07:00:40.614987906 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-16 07:01:24.597527259 +0000
@@ -32,58 +32,20 @@
    </title>
    <url>
     https://live-production.wcms.abc-cdn.net.au/6f61ef606af0bbbcbdaa7c6173a677e2?impolicy=wcms_crop_resize&amp;cropH=144&amp;cropW=144&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862
    </url>
    <link>
     https://www.abc.net.au/news/world/
    </link>
   </image>
   <item>
    <title>
-    Six mysterious 'Twin Peaks' deaths in Balkan Mountains stir controversy
-   </title>
-   <link>
-    https://www.abc.net.au/news/2026-02-15/bulgaria-twin-peaks-balkan-mountains-murders/106336984
-   </link>
-   <description>
-    Six unsolved deaths in two remote parts of the Balkan Mountains become a source of conspiracy across Bulgaria.
-   </description>
-   <dc:creator>
-    Brianna Morris-Grant
-   </dc:creator>
-   <pubDate>
-    Sun, 15 Feb 2026 01:45:53 +0000
-   </pubDate>
-   <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-15/bulgaria-twin-peaks-balkan-mountains-murders/106336984
-   </guid>
-   <category>
-    Homicide
-   </category>
-   <category>
-    Law, Crime and Justice
-   </category>
-   <media:group>
-    <media:description>
-     CCTV footage showing six people found dead in west Bulgarian mountains hugging each other and bidding farewell on the day of the killings.
-    </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1920&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1080&amp;xPos=420&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=1620&amp;xPos=150&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=491&amp;cropW=654&amp;xPos=643&amp;yPos=58&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=770&amp;cropW=433&amp;xPos=892&amp;yPos=65&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/c2684177413c1408884ee45ffb1826b8?impolicy=wcms_crop_resize&amp;cropH=1080&amp;cropW=810&amp;xPos=555&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-   </media:group>
-  </item>
-  <item>
-   <title>
     Gisèle Pelicot intends to visit ex-husband in prison to get answers
    </title>
    <link>
     https://www.abc.net.au/news/2026-02-15/gis%C3%A8le-pelicot-intends-to-visit-abuser-in-prison-to-get-answers/106346438
    </link>
    <description>
     Gisèle Pelicot, the woman at the centre of France's largest rape trial, says she intends to visit her ex-husband in jail and look her abuser "straight in the eye" to get answers to other allegations under investigation.
    </description>
    <dc:creator>
     Jessica Riga
@@ -173,12 +135,47 @@
     </media:description>
     <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=2444&amp;cropW=4345&amp;xPos=0&amp;yPos=236&amp;width=862&amp;height=485" width="862"/>
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4345&amp;cropW=4345&amp;xPos=0&amp;yPos=157&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=2897&amp;cropW=4345&amp;xPos=0&amp;yPos=236&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4827&amp;cropW=3620&amp;xPos=252&amp;yPos=173&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=3259&amp;cropW=4345&amp;xPos=0&amp;yPos=189&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4953&amp;cropW=2786&amp;xPos=787&amp;yPos=47&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/9300cec33a805dcc816ae48d509aab66?impolicy=wcms_crop_resize&amp;cropH=4827&amp;cropW=3620&amp;xPos=252&amp;yPos=173&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
+  <item>
+   <title>
+    Glove appearing to match Guthrie footage found as DNA tested
+   </title>
+   <link>
+    https://www.abc.net.au/news/2026-02-16/fbi-says-dna-recovered-from-glove-found-near-guthrie-home/106348058
+   </link>
+   <description>
+    The FBI says a glove containing DNA was found near the home of US TV show host Savannah Guthrie's mother and appears to match those worn by a masked person outside her front door the night she vanished.
+   </description>
+   <pubDate>
+    Sun, 15 Feb 2026 20:57:47 +0000
+   </pubDate>
+   <guid isPermaLink="true">
+    https://www.abc.net.au/news/2026-02-16/fbi-says-dna-recovered-from-glove-found-near-guthrie-home/106348058
+   </guid>
+   <category>
+    Law, Crime and Justice
+   </category>
+   <category>
+    Missing Person
+   </category>
+   <media:group>
+    <media:description>
+     The FBI has shared surveillance footage at the home of Nancy Guthrie the night she went missing in Tucson.
+    </media:description>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=830&amp;cropW=1476&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=732&amp;cropW=732&amp;xPos=744&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=984&amp;cropW=1476&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=984&amp;cropW=738&amp;xPos=738&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=984&amp;cropW=1312&amp;xPos=149&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=984&amp;cropW=554&amp;xPos=797&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/f0be1f1257c4ab2d23b76797d4954849?impolicy=wcms_crop_resize&amp;cropH=984&amp;cropW=738&amp;xPos=738&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+   </media:group>
+  </item>
  </channel>
 </rss>
