--- content_archive/ABC_News.xml	2026-02-11 19:00:08.572507167 +0000
+++ content_archive/ABC_News.xml.tmp	2026-02-11 19:00:30.420655390 +0000
@@ -70,124 +70,139 @@
     <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/992b09752e72b909227ac2a1e7bfdcd8?impolicy=wcms_crop_resize&amp;cropH=2809&amp;cropW=2809&amp;xPos=1363&amp;yPos=524&amp;width=862&amp;height=862" width="862"/>
     <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/992b09752e72b909227ac2a1e7bfdcd8?impolicy=wcms_crop_resize&amp;cropH=2337&amp;cropW=3506&amp;xPos=933&amp;yPos=618&amp;width=862&amp;height=575" width="862"/>
     <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/992b09752e72b909227ac2a1e7bfdcd8?impolicy=wcms_crop_resize&amp;cropH=2945&amp;cropW=2209&amp;xPos=1677&amp;yPos=273&amp;width=862&amp;height=1149" width="862"/>
     <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/992b09752e72b909227ac2a1e7bfdcd8?impolicy=wcms_crop_resize&amp;cropH=2599&amp;cropW=3466&amp;xPos=1153&amp;yPos=650&amp;width=862&amp;height=647" width="862"/>
     <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/992b09752e72b909227ac2a1e7bfdcd8?impolicy=wcms_crop_resize&amp;cropH=2830&amp;cropW=1592&amp;xPos=1845&amp;yPos=503&amp;width=862&amp;height=1533" width="862"/>
     <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/992b09752e72b909227ac2a1e7bfdcd8?impolicy=wcms_crop_resize&amp;cropH=2945&amp;cropW=2209&amp;xPos=1677&amp;yPos=273&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    FBI releases doorbell camera photos from Nancy Guthrie's home
+    Chinese flotilla sailed to edge of Australia's exclusive economic zone, says chief
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-11/fbi-release-pictures-of-masked-man-at-nancy-guthries-home/106329322
+    https://www.abc.net.au/news/2026-02-11/chinese-flotilla-edge-australian-exclusive-economic-zone/106333020
    </link>
    <description>
-    FBI director Kash Patel has posted the images on social media, saying they had been recovered after days of work by investigators.
+    Under questioning in Senate estimates, the chief of the Australian Defence Force reveals that a Chinese flotilla sailed within seven nautical miles of Australia's 200 nautical mile exclusive economic zone.
    </description>
+   <dc:creator>
+    Stephen Dziedzic and Olivia Caisley
+   </dc:creator>
    <pubDate>
-    Tue, 10 Feb 2026 18:19:56 +0000
+    Wed, 11 Feb 2026 07:55:04 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-11/fbi-release-pictures-of-masked-man-at-nancy-guthries-home/106329322
+    https://www.abc.net.au/news/2026-02-11/chinese-flotilla-edge-australian-exclusive-economic-zone/106333020
    </guid>
    <category>
-    Crime
+    Defence and National Security
+   </category>
+   <category>
+    Government and Politics
    </category>
    <category>
-    Missing Person
+    World Politics
+   </category>
+   <category>
+    Foreign Affairs
    </category>
    <media:group>
     <media:description>
-     A video still from the doorbell camera of Nancy Guthrie's home that has been released by the FBI.
+     David Johnston says a Chinese flotilla stayed "just clear" of Australia's 200 nautical mile exclusive economic zone.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=864&amp;cropW=1536&amp;xPos=0&amp;yPos=40&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=1536&amp;cropW=1536&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=1024&amp;cropW=1536&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=2048&amp;cropW=1536&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=1152&amp;cropW=1536&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=2048&amp;cropW=1152&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/df551f4eca29ee657185f8d8e5b7dfec?impolicy=wcms_crop_resize&amp;cropH=2048&amp;cropW=1536&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=2813&amp;cropW=5000&amp;xPos=0&amp;yPos=137&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=3333&amp;xPos=834&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=5000&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1176&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=4444&amp;xPos=278&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=1875&amp;xPos=1563&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/d1ed91915abefdee519420125d0f850a?impolicy=wcms_crop_resize&amp;cropH=3333&amp;cropW=2500&amp;xPos=1176&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    'We are only getting started,' ICE boss vows at fiery hearing
+    Indonesia to send up to 8,000 troops to Gaza under Trump's Board of Peace
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-11/ice-customs-border-patrol-heads-questioned-by-congress/106329244
+    https://www.abc.net.au/news/2026-02-11/indonesia-to-send-8000-troops-to-gaza-idf-peacekeeping/106330718
    </link>
    <description>
-    Democrats level a litany of complaints against the heads of America's immigration agencies during a rare hearing in Washington.
+    The move makes Indonesia the first country to clearly commit troops to a multinational peacekeeping effort in the war-torn enclave.
    </description>
    <dc:creator>
-    Brad Ryan
+    Tim Swanston and Hellena Souisa
    </dc:creator>
    <pubDate>
-    Tue, 10 Feb 2026 21:16:03 +0000
+    Wed, 11 Feb 2026 03:29:05 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-11/ice-customs-border-patrol-heads-questioned-by-congress/106329244
+    https://www.abc.net.au/news/2026-02-11/indonesia-to-send-8000-troops-to-gaza-idf-peacekeeping/106330718
    </guid>
    <category>
+    Unrest, Conflict and War
+   </category>
+   <category>
     World Politics
    </category>
    <category>
-    Immigration
+    Foreign Affairs
+   </category>
+   <category>
+    Defence Forces
+   </category>
+   <category>
+    Territorial Disputes
    </category>
    <category>
-    Migration Policy
+    Foreign Aid
    </category>
    <media:group>
     <media:description>
-     Rodney Scott, Joseph Edlow and Todd Lyons were questioned over how their immigration agencies were operating.
+     A spokesperson for Indonesia's President Prabowo Subianto (right) says the country is willing to send up to 8,000 troops.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2250&amp;cropW=4000&amp;xPos=0&amp;yPos=287&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2666&amp;cropW=2666&amp;xPos=1316&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2666&amp;cropW=3999&amp;xPos=1&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2666&amp;cropW=1999&amp;xPos=382&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2666&amp;cropW=3555&amp;xPos=276&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2666&amp;cropW=1500&amp;xPos=1250&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/39ad22255b0ba48599cb2893a2ee183a?impolicy=wcms_crop_resize&amp;cropH=2666&amp;cropW=1999&amp;xPos=382&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=514&amp;cropW=914&amp;xPos=317&amp;yPos=135&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=775&amp;cropW=775&amp;xPos=383&amp;yPos=16&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=514&amp;cropW=771&amp;xPos=374&amp;yPos=129&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=775&amp;cropW=581&amp;xPos=480&amp;yPos=16&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=576&amp;cropW=768&amp;xPos=385&amp;yPos=121&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=775&amp;cropW=436&amp;xPos=552&amp;yPos=16&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/416195a1d35d9fdba9aaa85eaa8a1ba0?impolicy=wcms_crop_resize&amp;cropH=775&amp;cropW=581&amp;xPos=480&amp;yPos=16&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
   <item>
    <title>
-    'My features were a puzzle': Surgeon Kelvin Kong on growing up Chinese and Indigenous
+    US statement accuses Palau senate president of taking bribes from China
    </title>
    <link>
-    https://www.abc.net.au/news/2026-02-11/kelvin-kong-s-growing-up-chinese-and-indigenous-australia/106326388
+    https://www.abc.net.au/news/2026-02-11/us-bars-palau-marshall-islands-officials-for-alleged-corruption/106329288
    </link>
    <description>
-    Australia's first identified First Nations surgeon, Dr Kelvin Kong spoke about his Malaysian Chinese heritage during the annual Wang Gungwu lecture at Parliament House.
+    A senior Palau politician rejects accusations from the United States Department of State that he accepted bribes from "government, business, and criminal interests from China".
    </description>
    <dc:creator>
-    Iris Zhao
+    Patrick Martin and Evan Wasuka
    </dc:creator>
    <pubDate>
-    Tue, 10 Feb 2026 20:44:10 +0000
+    Wed, 11 Feb 2026 07:30:57 +0000
    </pubDate>
    <guid isPermaLink="true">
-    https://www.abc.net.au/news/2026-02-11/kelvin-kong-s-growing-up-chinese-and-indigenous-australia/106326388
+    https://www.abc.net.au/news/2026-02-11/us-bars-palau-marshall-islands-officials-for-alleged-corruption/106329288
    </guid>
    <category>
-    Indigenous Australians
-   </category>
-   <category>
-    Multiculturalism
+    World Politics
    </category>
    <media:group>
     <media:description>
-     Surgeon Kelvin Kong has a passion for ear health and treating the epidemic levels of ear infections in Indigenous children.
+     Palau has a Compact of Free Association with the United States.
     </media:description>
-    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=926&amp;cropW=1646&amp;xPos=0&amp;yPos=19&amp;width=862&amp;height=485" width="862"/>
-    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=1161&amp;cropW=1161&amp;xPos=250&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
-    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=1097&amp;cropW=1646&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
-    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=1161&amp;cropW=871&amp;xPos=395&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
-    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=1161&amp;cropW=1548&amp;xPos=56&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
-    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=1161&amp;cropW=653&amp;xPos=504&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
-    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/a6e6eb4ad42ab3c33ecf510a2d7688e2?impolicy=wcms_crop_resize&amp;cropH=1161&amp;cropW=871&amp;xPos=395&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="485" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1407&amp;cropW=2501&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=485" width="862"/>
+    <media:content height="862" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1751&amp;cropW=1751&amp;xPos=375&amp;yPos=0&amp;width=862&amp;height=862" width="862"/>
+    <media:content height="575" isDefault="true" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1668&amp;cropW=2501&amp;xPos=0&amp;yPos=0&amp;width=862&amp;height=575" width="862"/>
+    <media:content height="1149" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1751&amp;cropW=1312&amp;xPos=594&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
+    <media:content height="647" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1751&amp;cropW=2334&amp;xPos=83&amp;yPos=0&amp;width=862&amp;height=647" width="862"/>
+    <media:content height="1533" medium="image" type="image/jpeg" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1751&amp;cropW=984&amp;xPos=759&amp;yPos=0&amp;width=862&amp;height=1533" width="862"/>
+    <media:thumbnail height="1149" url="https://live-production.wcms.abc-cdn.net.au/f8f4f552b68f46d6ad839b3dd2c9f5ef?impolicy=wcms_crop_resize&amp;cropH=1751&amp;cropW=1312&amp;xPos=594&amp;yPos=0&amp;width=862&amp;height=1149" width="862"/>
    </media:group>
   </item>
  </channel>
 </rss>
