--- previous+++ current@@ -1,5 +1,7 @@ - 
-Plant breeders rights ApplyNew PBR Qualified Persons portal – Release postponed 19 June 2026 The planned launch of the Qualified Persons (QP) portal for 20 June 2026 has been postponed.
+Plant breeders rights ApplyNew Qualified Persons (QP) portal launching 7 July 2026 02 July 2026 IP Australia is introducing a new Qualified Persons (QP) portal to replace the current Interactive Variety Description System (IVDS).
+- 
+Patents ApplyPatent Cooperation Treaty Fees Change – 1 July 2026 01 July 2026 | 2 MIN READ In accordance with the directives adopted by the PCT Assembly during the month of October of each year, new equivalent amounts in AUD will be established with effect from 1 July 2026.
 - 
 Trade marksPractice update following Zip Co Ltd v Firstmac Ltd 19 June 2026 The Trade Marks Manual of Practice and Procedure has been updated following the High Court's decision in Zip Co Limited v Firstmac Limited.
 - 
@@ -16,6 +18,4 @@ PatentsOtsuka Pharmaceutical Co Ltd v Sun Pharma ANZ Pty Ltd - Update 18 March 2026 | 2 MIN READ IP Australia paused processing of all patent extension of term applications while we considered the Full Federal Court’s decision in Otsuka Pharmaceutical Co Ltd v Sun Pharma ANZ Pty Ltd [2025] FCAFC 161.
 - 
 PatentsConsultation on Patent Manual update on computer-implemented inventions 16 March 2026 | 2 MIN READ We're undertaking additional consultation on updates to the Patent Manual of Practice and Procedure.
-- 
-Trade marks ApplyUpdate to trade mark image indexing terminology 11 March 2026 | 2 MIN READ IP Australia has updated some of the terminology used to index images in trade mark applications to reflect respectful, contemporary language.
- Showing 40 results+ Showing 42 results