--- previous+++ current@@ -1,7 +1,9 @@+- 
+Design rightsUpdated design instruments 24 August 2026 IP Australia has updated instruments that relate to the paying of fees, the general filing of documents, and the filing of documents and evidence during opposition and other proceedings.
 - 
 Trade marks Plant breeders rightsExamination of trade mark applications relating to plants or plant materials 13 August 2026 IP Australia has updated the Trade Marks Manual of Practice and Procedure with revised guidance on examining trade mark applications relating to plants or plant materials.
 - 
-Plant breeders rights ApplyNew PBR Part 1 application form coming soon 12 August 2026 IP Australia will introduce a new electronic Plant Breeder's Rights (PBR) Part 1 application form in early September 2026.
+Plant breeders rights ApplyNew PBR Part 1 application form coming soon 12 August 2026 IP Australia will introduce a new electronic Plant Breeder's Rights (PBR) Part 1 application form on Wednesday 9 September 2026.
 - 
 PatentsUpdated patent instruments 07 August 2026 IP Australia has updated instruments that relate to the paying of fees, the general filing of documents, and the filing of documents and evidence during opposition and other proceedings.
 - 
@@ -16,6 +18,4 @@ Trade marksTrade Marks Office Manual of Practice and Procedure 13 July 2026 IP Australia has updated the Trade Marks Manual of Practice and Procedure with a new chapter on filing requirements.
 - 
 Design rightsSurrender of Registration 09 July 2026 A request has been made under Section 49 of the Designs Act 2003 to surrender Design Registration Nos. 202519552, 202519553, 202519554 and 2025195555.
-- 
-PatentsConsultation response and updated Patent Manual release 03 July 2026 Updated guidance in Patent Manual of Practice and Procedures on computer-implemented inventions.
- Showing 52 results+ Showing 53 results