--- previous+++ current@@ -1,3 +1,7 @@+- 
+Design rightsSurrender of Registration 09 July 2026 A request has been made under Section 49 of the Designs Act 2003 to surrender Design Registration Nos. 202519552, 202519553, 202519554 and 2025195555.
+- 
+PatentsConsultation response and updated Patent Manual release 03 July 2026 Updated guidance in Patent Manual of Practice and Procedures on computer-implemented inventions.
 - 
 Plant breeders rights ApplyNew Qualified Persons (QP) portal launching 7 July 2026 02 July 2026 IP Australia is introducing a new Qualified Persons (QP) portal to replace the current Interactive Variety Description System (IVDS).
 - 
@@ -14,8 +18,4 @@ Trade marksSigns newly notified under Article 6ter of the Paris Convention published 31 March 2026 08 May 2026 | 2 MIN READ Since 2009, newly notified signs have been published online by the World Intellectual Property Organization (WIPO) every 6 months at the end of March and September.
 - 
 Trade marksBulk data product update 05 May 2026 | 2 MIN READ In late March 2026, we identified a technical coding issue affecting the trade mark bulk data product we provide to third party subscribers.
-- 
-PatentsOtsuka Pharmaceutical Co Ltd v Sun Pharma ANZ Pty Ltd - Update 18 March 2026 | 2 MIN READ IP Australia paused processing of all patent extension of term applications while we considered the Full Federal Court’s decision in Otsuka Pharmaceutical Co Ltd v Sun Pharma ANZ Pty Ltd [2025] FCAFC 161.
-- 
-PatentsConsultation on Patent Manual update on computer-implemented inventions 16 March 2026 | 2 MIN READ We're undertaking additional consultation on updates to the Patent Manual of Practice and Procedure.
- Showing 42 results+ Showing 44 results