--- previous+++ current@@ -1,4 +1,8 @@ -
+Trade marks
+Update: Zip Co Ltd v Firstmac Ltd
+18 May 2026 | 1 MIN READ
+The Registrar of Trade Marks is assessing the implications of the decision in Zip Co Ltd v Firstmac Ltd (S140-2025) [2026] HCA 16. -
 Olympic Insignia Protection (Approved Forms) Instrument 2026
 10 May 2026 | 1 MIN READ
 The Registrar of Designs has published approved forms for Olympic designs, pursuant to section 14 of the Olympic Insignia Protection Act 1987. -
@@ -31,9 +35,5 @@ In accordance with the directives adopted by the PCT Assembly during the month of October of each year, new equivalent amounts in AUD will be established with effect from 1 January 2026. -
 Declaration of Public Holidays for 2026
 04 December 2025
-During the period 1 January 2026 to 1 January 2027, the Canberra office will not be open for business on these days. -
-Trade marks
-Trade Marks Amendment (International Registrations, Hearings and Oppositions) Regulations 2025
-17 November 2025
-The Trade Marks Amendment (International Registrations, Hearings and Oppositions) Regulations 2025 (the IRHO Regulations) amend the Trade Marks Regulations 1995.
-Showing 36 results+During the period 1 January 2026 to 1 January 2027, the Canberra office will not be open for business on these days.
+Showing 37 results