--- previous+++ current@@ -1,4 +1,8 @@ -
+Trade marks
+Bulk data product update
+05 May 2026 | 2 MIN READ
+In late March 2026, we identified a technical coding issue affecting the trade mark bulk data product we provide to third party subscribers. -
 Patents
 Otsuka Pharmaceutical Co Ltd v Sun Pharma ANZ Pty Ltd - Update
 18 March 2026 | 2 MIN READ
@@ -31,8 +35,5 @@ Patents
 Commissioner of Patents guidance update regarding computer-implemented inventions
 14 October 2025
-On 14 October 2025, IP Australia updated the guidelines in our Patent Manual of Practice and Procedures and has sought special leave to appeal to the High Court of Australia. -
-Signs newly notified under Article 6ter of the Paris Convention – published 31 March 2025
-18 September 2025
-The World Intellectual Property Organization (WIPO) has published newly notified signs online.
-Showing 33 results+On 14 October 2025, IP Australia updated the guidelines in our Patent Manual of Practice and Procedures and has sought special leave to appeal to the High Court of Australia.
+Showing 34 results