--- previous+++ current@@ -1,3 +1,7 @@+- 
+Design rights AmendCorrection of entry in the Designs Register under reg 9.05(2)(a) 06 August 2026 The Registrar has approved an amendment to Design Registration No. 202611073 to correct the representations published in the Designs Register.
+- 
+Design rights AmendCorrection of entry in the Designs Register under reg 9.05(2)(a) 202615130 23 July 2026 This notice approves an amendment to Design Registration No. 202615130 to correct the published design representations.
 - 
 Trade marksTrade Marks Office Manual of Practice and Procedure 13 July 2026 IP Australia has updated the Trade Marks Manual of Practice and Procedure with a new chapter on filing requirements.
 - 
@@ -9,13 +13,9 @@ - 
 Patents ApplyPatent Cooperation Treaty Fees Change – 1 July 2026 01 July 2026 | 2 MIN READ In accordance with the directives adopted by the PCT Assembly during the month of October of each year, new equivalent amounts in AUD will be established with effect from 1 July 2026.
 - 
+Design rightsSurrender of registration 202014719 and 202116466 29 June 2026 A request under Section 49 of the Designs Act 2003 has been made by the registered owner of the Design Registration Nos 202014719 and 202116466 to surrender the registration.
+- 
 Trade marksPractice update following Zip Co Ltd v Firstmac Ltd 19 June 2026 The Trade Marks Manual of Practice and Procedure has been updated following the High Court's decision in Zip Co Limited v Firstmac Limited.
 - 
 PatentsUpdates to patent examination technology areas 08 June 2026 Technology area groups updated to better connect customers with the right expertise.
-- 
-Trade marksUpdate: Zip Co Ltd v Firstmac Ltd 18 May 2026 | 1 MIN READ The Registrar of Trade Marks is assessing the implications of the decision in Zip Co Ltd v Firstmac Ltd (S140-2025) [2026] HCA 16.
-- 
-Olympic Insignia Protection (Approved Forms) Instrument 2026 10 May 2026 | 1 MIN READ The Registrar of Designs has published approved forms for Olympic designs, pursuant to section 14 of the Olympic Insignia Protection Act 1987.
-- 
-Trade marksSigns newly notified under Article 6ter of the Paris Convention published 31 March 2026 08 May 2026 | 2 MIN READ Since 2009, newly notified signs have been published online by the World Intellectual Property Organization (WIPO) every 6 months at the end of March and September.
- Showing 45 results+ Showing 47 results