--- previous+++ current@@ -1,4 +1,11 @@ -
+Olympic Insignia Protection (Approved Forms) Instrument 2026
+10 May 2026 | 1 MIN READ
+The Registrar of Designs has published approved forms for Olympic designs, pursuant to section 14 of the Olympic Insignia Protection Act 1987. -
+Trade marks
+Signs newly notified under Article 6ter of the Paris Convention published 31 March 2026
+08 May 2026 | 2 MIN READ
+Since 2009, newly notified signs have been published online by the World Intellectual Property Organization (WIPO) every 6 months at the end of March and September. -
 Trade marks
 Bulk data product update
 05 May 2026 | 2 MIN READ
@@ -28,12 +35,5 @@ Trade marks
 Trade Marks Amendment (International Registrations, Hearings and Oppositions) Regulations 2025
 17 November 2025
-The Trade Marks Amendment (International Registrations, Hearings and Oppositions) Regulations 2025 (the IRHO Regulations) amend the Trade Marks Regulations 1995. -
-Signs newly notified under Article 6ter of the Paris Convention published – 30 September 2025
-16 October 2025
-The World Intellectual Property Organization (WIPO) has published newly notified signs online. -
-Patents
-Commissioner of Patents guidance update regarding computer-implemented inventions
-14 October 2025
-On 14 October 2025, IP Australia updated the guidelines in our Patent Manual of Practice and Procedures and has sought special leave to appeal to the High Court of Australia.
-Showing 34 results+The Trade Marks Amendment (International Registrations, Hearings and Oppositions) Regulations 2025 (the IRHO Regulations) amend the Trade Marks Regulations 1995.
+Showing 36 results