--- previous+++ current@@ -9,7 +9,6 @@ - a basic fee (653 Swiss francs; or 903 Swiss francs for a mark in color) (Note! A 90% reduction applies if you are filing an application in the IP Office of a least-developed country); and
 - additional fees depending on where you want to protect your mark, and how many classes of goods and services you want to cover.
 To expand the geographical coverage of, or modify or renew your international registration you will have to pay additional fees.
- 
 Find out more
 The Madrid System cuts through the complexity of securing global trademark protection in multiple markets. You don’t need to navigate different filing procedures, languages and currencies in each country. Instead, you get a single international trademark registration that covers all the markets where protection is granted. (The intellectual property (IP) office of each member where you seek protection will decide whether it can grant protection under its own domestic laws.)
 - One registration to manage; one renewal date
@@ -52,6 +51,12 @@ WIPO Brands and Designs
 Madrid System education resources
 Public Madrid System webinars
+Madrid System success stories
+Madrid System podcast
+Learn about the History of the Madrid System in our podcast series "WIPOD – International Trademark System Talks".
+Want to know more? Try our Madrid System Distance Learning course.
+Madrid System education resources
+Public Madrid System webinars
 Madrid System In-Person Seminar
 Madrid System success stories
 Madrid System podcast
