--- previous+++ current@@ -4,7 +4,7 @@ - Protection in China does not include Hong Kong or Macao.
 - Protection in Denmark includes Greenland and Faroe Islands.
 - Protection in France includes French Guiana, French Polynesia, French Southern and Antarctic Lands, Guadeloupe, Martinique, Mayotte, New Caledonia, Réunion, Saint Barthélemy, Saint Martin (French part), Saint Pierre and Miquelon, as well as Wallis and Futuna.
-- Protection in the United Kingdom includes England, Wales, Scotland, Northern Ireland, the Isle of Man, Bailiwick of Jersey (until August 1, 2026), Gibraltar, and the British Overseas Territory of the Falkland Islands (Malvinas).
+- Protection in the United Kingdom includes England, Wales, Scotland, Northern Ireland, the Isle of Man, Gibraltar, and the British Overseas Territory of the Falkland Islands (Malvinas).
 Fees include:
 - a basic fee (653 Swiss francs; or 903 Swiss francs for a mark in color) (Note! A 90% reduction applies if you are filing an application in the IP Office of a least-developed country); and
 - additional fees depending on where you want to protect your mark, and how many classes of goods and services you want to cover.
@@ -43,9 +43,11 @@ - Madrid System pendency rates [PDF]
 - Madrid Yearly Review
 - 
-Q3 2026 Madrid System Webinars: Join Us Online July 28, 2026 
+Transition to Madrid e-Filing at the USPTO July 31, 2026
 - 
-Saudi Arabia Joins WIPO’s Madrid System July 23, 2026 
+Q3 2026 Madrid System Webinars: Join Us Online July 28, 2026
+- 
+Saudi Arabia Joins WIPO’s Madrid System July 23, 2026
 Madrid System news
 WIPO Brands and Designs
 Madrid System education resources
@@ -53,4 +55,5 @@ Madrid System In-Person Seminar
 Madrid System success stories
 Madrid System podcast
+Learn about the History of the Madrid System in our podcast series "WIPOD – International Trademark System Talks".
 Want to know more? Try our Madrid System Distance Learning course.