--- previous+++ current@@ -1,6 +1,6 @@ Information: Our website will be undergoing an upgrade on September 23, 2025, from 6.30 a.m. to 8.00 a.m. Geneva time (GMT+2). While some pages may be momentarily unavailable, IP tools will remain available.
 What is the Madrid System?
- ____________________________________________________________________
+____________________________________________________________________
 - Protection in China does not include Hong Kong or Macao.
 - Protection in Denmark includes Greenland and Faroe Islands.
 - Protection in France includes French Guiana, French Polynesia, French Southern and Antarctic Lands, Guadeloupe, Martinique, Mayotte, New Caledonia, Réunion, Saint Barthélemy, Saint Martin (French part), Saint Pierre and Miquelon, as well as Wallis and Futuna.
@@ -41,12 +41,8 @@ - Madrid System statistics
 - Madrid System pendency rates [PDF]
 - Madrid Yearly Review
-- 
-Helping SMEs Protect Brands and Designs in Latin America and the Caribbean September 14, 2026
-- 
-One Year of eMadrid September 9, 2026
-- 
-Transition to Madrid e-Filing at the USPTO July 31, 2026
+- HURRAW! Natural Lip Balms: From Main Street to Global Brand October 7, 2026
+- WIPO Wallet Is Now Live! October 5, 2026
 Madrid System news
 WIPO Brands and Designs
 Madrid System education resources
