--- previous+++ current@@ -1,5 +1,10 @@ Information: Our website will be undergoing an upgrade on September 23, 2025, from 6.30 a.m. to 8.00 a.m. Geneva time (GMT+2). While some pages may be momentarily unavailable, IP tools will remain available.
 What is the Madrid System?
+ ____________________________________________________________________
+- Protection in China does not include Hong Kong or Macao.
+- Protection in Denmark includes Greenland and Faroe Islands.
+- Protection in France includes French Guiana, French Polynesia, French Southern and Antarctic Lands, Guadeloupe, Martinique, Mayotte, New Caledonia, Réunion, Saint Barthélemy, Saint Martin (French part), Saint Pierre and Miquelon, as well as Wallis and Futuna.
+- Protection in the United Kingdom includes England, Wales, Scotland, Northern Ireland, the Isle of Man, Bailiwick of Jersey (until August 1, 2026), Gibraltar, and the British Overseas Territory of the Falkland Islands (Malvinas).
 Fees include:
 - a basic fee (653 Swiss francs; or 903 Swiss francs for a mark in color) (Note! A 90% reduction applies if you are filing an application in the IP Office of a least-developed country); and
 - additional fees depending on where you want to protect your mark, and how many classes of goods and services you want to cover.
@@ -37,6 +42,10 @@ - Madrid System statistics
 - Madrid System pendency rates [PDF]
 - Madrid Yearly Review
+- 
+WIPO Brings Global IP Expertise to VivaTech 2026 June 25, 2026 
+- 
+Procedural Changes Coming Up for Jersey, UK June 23, 2026 
 Madrid System news
 WIPO Brands and Designs
 Madrid System education resources
