--- previous+++ current@@ -1,4 +1,14 @@-11500 results, showing 1 to 20
+11507 results, showing 1 to 20
+ 
+ On 12 August 2026, the ACCC announced its decision to approve Black Rhino Group Pty Ltd via its subsidiaries Black Rhino (Club Roma) Pty Ltd and Light Them Up (Roma) Pty Ltd as trustee for Light Them Up (Roma) Trust’s acquisition of Club Hotel Motel Roma, Cellarbrations at Superstore Roma, and The Northern Road Bottle Shop/Bottle-O subject to conditions.
+ 
+ On 10 July 2026, Australia Post provided the ACCC with a price notification proposing price increases for its reserved ordinary letter services to apply from 1 September 2026.
+ 
+ The Australian Competition and Consumer Commission (ACCC) has accepted a court enforceable undertaking from Vacaciones eDreams, S.L.
+ 
+ The Australian Competition and Consumer Commission (ACCC) has accepted a court enforceable undertaking from Marshall Freeman Collections Pty Ltd (MFC) in relation to undue harassment, misleading or deceptive conduct and false or misleading representations in the course of debt collection activities that contravened, or likely contravened, sections 18, 29(1)(m) and 50(1)(a) of the Australian Consumer Law (ACL).
+ 
+ On 31 July 2026, the ACCC announced its decision to approve Heidelberg Materials Australia Holdings Pty Ltd.’s acquisition of Maas Group Holdings’ construction materials business, subject to conditions that it divest three concrete plants and a quarry across Queensland and New South Wales (together, the Divestiture Businesses).
  
  Status: Exemption granted for 10 years
  
@@ -10,11 +20,4 @@  
  The Australian Competition and Consumer Commission (ACCC) has accepted a court enforceable undertaking from EnergyAustralia Pty Ltd (EnergyAustralia) in relation to its failure to make available to residential customers a Solar Sharer Offer by 1 July 2026.
  
- The Australian Competition and Consumer Commission declares, pursuant to section 152AL(3) of the Competition and Consumer Act 2010 (Cth) (the Act), that the superfast broadband access service (SBAS) is a "declared service" for the purposes of Part XIC of the Act.
-Status: Exemption granted for 10 years
- 
- Status: Exemption granted for 10 years (excluding Hay South NSW)
- 
- On 30 June 2026, the ACCC granted exemptions under section 56GD of the Competition and Consumer Act 2010 (Cth) to Metrics Credit Partners Pty Ltd, Devizo Finance Pty Ltd, MBF Leasing Pty Ltd, MBF SME Lending Pty Ltd, Bigstone Lending Pty Ltd, QT Investment Management Pty Ltd, BC Invest Loans Pty Ltd, Taurus Finance Holdings Pty Ltd and Mortgageport Home Loans Pty Ltd, deferring each entity’s product data sharing obligations until 30 June 2027, and each entity’s consumer data sharing obligations until 30 June 2028.
- 
- On 30 June 2026, the Australian Competition and Consumer Commission (ACCC) granted an exemption under section 56GD of the Competition and Consumer Act 2010 (Cth) to De Lage Landen Pty Limited (DLL) and AGCO Finance Pty Limited (AGCO Finance) deferring each entity’s consumer data sharing obligations until 31 December 2029.+ The Australian Competition and Consumer Commission declares, pursuant to section 152AL(3) of the Competition and Consumer Act 2010 (Cth) (the Act), that the superfast broadband access service (SBAS) is a "declared service" for the purposes of Part XIC of the Act.