--- previous+++ current@@ -1,26 +1,26 @@-Note: The fees listed below are effective from 1 July 2025.
+Note: The fees listed below are effective from 1 July 2026.
 Cheques or Money orders for the payment of fees should be made payable to the Federal Court of Australia.
 A number of fees apply to Federal Circuit and Family Court of Australia (Division 2) proceedings pursuant to Schedule 1 Part 2 of the Federal Court and Federal Circuit and Family Court Regulations 2022.
-The current fees are contained within the gazetted Annual federal courts and tribunals fee increases from 1 July 2025 or Federal Court and Federal Circuit and Family Court Amendment (Filing Fees) Regulations 2026.
+The current fees are contained within the gazetted Annual federal courts and tribunals fee increases from 1 July 2026 or Federal Court and Federal Circuit and Family Court Amendment (Filing Fees) Regulations 2026.
 | Document or service | Fee | |
 |---|---|---|
 | Corporations | Others | |
-| Filing an application (201) ^ Note: this is not the fee payable for migration applications, see below | $2,005 | $835 | 
-| Filing an application (migration only) (201A) | (a) full fee—$4,015, or (b) if a Registrar or an authorised officer has determined that the person may pay a reduced fee under section 2.06A—$2,005 | |
-| Response* Seeking different orders sought by applicant (207) | $2,005 | $825 | 
-| Interlocutory – interim (204)* | $1,205 | $485 | 
-| Setting down for hearing fee (215)* | $2,395 | $995 | 
-| Daily Hearing Fee* (for each hearing day, or part of a hearing day excluding the first hearing day) (216) | $2,395 | $995 | 
-| Mediation by a court officer* (For each attendance) (224) | $665 | |
-| Filing an Application for a Review of a Registrar's Decision (205)* | $1,205 | $485 | 
-| For the hearing of an application (including a cross-claim) under subsection 256(2) of the Federal Circuit and Family Court Act – for each day or part of day (218)* | $2,395 | $995 | 
-| Filing a bill of costs (203)* | $325 | |
-| Issuing a subpoena (222)* | $190 | $95 | 
+| Filing an application (201) ^ Note: this is not the fee payable for migration applications, see below | $2,085 | $870 | 
+| Filing an application (migration only) (201A) | (a) full fee—$4,180, or (b) if a Registrar or an authorised officer has determined that the person may pay a reduced fee under section 2.06A—$2,090 | |
+| Response* Seeking different orders sought by applicant (207) | $2,085 | $860 | 
+| Interlocutory – interim (204)* | $1,255 | $505 | 
+| Setting down for hearing fee (215)* | $2,495 | $1,035 | 
+| Daily Hearing Fee* (for each hearing day, or part of a hearing day excluding the first hearing day) (216) | $2,495 | $1,035 | 
+| Mediation by a court officer* (For each attendance) (224) | $690 | |
+| Filing an Application for a Review of a Registrar's Decision (205)* | $1,255 | $505 | 
+| For the hearing of an application (including a cross-claim) under subsection 256(2) of the Federal Circuit and Family Court Act – for each day or part of day (218)* | $2,495 | $1,035 | 
+| Filing a bill of costs (203)* | $340 | |
+| Issuing a subpoena (222)* | $200 | $100 | 
 | On request, production of file and making of a copy or copies of a document or documents in the file: (219) | ||
 | Production of the file | $60 | |
 | For each page included in a copy made in accordance with the request | $1 | |
-| Seizure and sale of goods (221) | $765 | |
-| Filing an application to register a New Zealand judgment under the Trans-Tasman Proceedings Act 2010 (214A) | $155 | |
+| Seizure and sale of goods (221) | $795 | |
+| Filing an application to register a New Zealand judgment under the Trans-Tasman Proceedings Act 2010 (214A) | $160 | |
 * Note: this fee is not payable for human rights applications, some fair work applications, and small claims proceedings in fair work and consumer credit claims.
 Service and execution
 For each service or execution, or attempted service or execution, of the process of the Federal Circuit and Family Court of Australia (Division 2) by an officer of the court** an amount equal to:
@@ -35,29 +35,29 @@ | Document or service | Fee | |
 |---|---|---|
 | If the claim is less than $10,000 or all claims are seeking a non-monetary remedy (213) | If the claim is between $10,000 and $40,000 (214) | |
-| Filing of an application under Consumer Credit Protection Act 2009 if the applicant indicates they want the small claims procedure to apply | $300 | $480 | 
+| Filing of an application under Consumer Credit Protection Act 2009 if the applicant indicates they want the small claims procedure to apply | $310 | $500 | 
 Industrial Law
 For proceedings other than the following the general application fee applies:
 Fair work
 | Document or service | Fee | 
 |---|---|
 | Filing of an application under section 539 of the Fair Work Act 2009 in either of the following circumstances: 
- | $89.70 | 
-| Filing of an application under section 539 of the Fair Work Act 2009 for orders in relation to an alleged contravention of Division 2 of Part 3‑5A of that Act (209A) | $90.00 | 
-| Filing of an application under section 539 of the Fair Work Act 2009 if the applicant has been dismissed from employment in alleged contravention of section 772 of that Act (210) | $89.70 | 
+ | $92.70 | 
+| Filing of an application under section 539 of the Fair Work Act 2009 for orders in relation to an alleged contravention of Division 2 of Part 3‑5A of that Act (209A) | $95.00 | 
+| Filing of an application under section 539 of the Fair Work Act 2009 if the applicant has been dismissed from employment in alleged contravention of section 772 of that Act (210) | $92.70 | 
 Small claims
 | Document or service | Fee | |||
 |---|---|---|---|---|
 | If the claim is less than $10,000 (211) | If the claim is between $10,000 and $50,000 (212) | If the claim is between $50,000 and $100,000 (212AA) | and the proceedings relate to one or more of the matters mentioned in paragraph 548(1B)(a) of that Act (other than a proceeding mentioned in item 212) (212A) | |
-| Filing of an application under section 548 of the Fair Work Act 2009 if the applicant indicates they want the small claims procedure to apply | $300 | $480 | $560 | $300 | 
+| Filing of an application under section 548 of the Fair Work Act 2009 if the applicant indicates they want the small claims procedure to apply | $310 | $500 | $585 | $310 | 
 Bankruptcy
 | Document or service | Fee | |||
 |---|---|---|---|---|
 | Publicly listed company | Corporations | Public authority | Others | |
-| Filing an application under the Bankruptcy Act 1966 (206) | $7,035 | $4,695 | $4,695 | $1,955 | 
-| Application for substituted service of a bankruptcy notice (208) | $790 | $520 | $520 | $205 | 
-| For issuing a summons to a person, under section 50 or 81 of the Bankruptcy Act 1966 to attend examination about a debtor's examinable affairs (223) | $975 | $630 | $630 | $325 | 
-| Examination by a Registrar under section 50 or 81 of the Bankruptcy Act 1966 – for each day or part of a day (217) | $3,625 | $2,395 | $2,395 | $995 | 
+| Filing an application under the Bankruptcy Act 1966 (206) | $7,325 | $4,885 | $4,885 | $2,035 | 
+| Application for substituted service of a bankruptcy notice (208) | $820 | $540 | $540 | $215 | 
+| For issuing a summons to a person, under section 50 or 81 of the Bankruptcy Act 1966 to attend examination about a debtor's examinable affairs (223) | $1,015 | $655 | $655 | $340 | 
+| Examination by a Registrar under section 50 or 81 of the Bankruptcy Act 1966 – for each day or part of a day (217) | $3,775 | $2,495 | $2,495 | $1,035 | 
 Human rights
 | Document or service | Fee | 
 |---|---|
@@ -65,7 +65,7 @@ Migration
 | Document or service | Fee | 
 |---|---|
-| Filing an application (migration only) (201A) | (a) full fee—$4,015, or (b) if a Registrar or an authorised officer has determined that the person may pay a reduced fee under section 2.06A—$2,005 | 
+| Filing an application (migration only) (201A) | (a) full fee—$4,180, or (b) if a Registrar or an authorised officer has determined that the person may pay a reduced fee under section 2.06A—$2,090 | 
 Party Category and Fee Calculation
 Section 2.02 of the Regulation read in conjunction with the definitions in sections 1.03 and 1.04 sets out the basis of how fees are to be calculated. Fees for filing documents, setting down, for hearing, for mediation and for some services are calculated according to the type of body or person liable to pay them. A publicly listed company pays the highest rate of fees. A corporation and a public authority pay the next highest rate. All others, including any individuals, small businesses or unincorporated not-for-profit association, pay the lowest rate.
 So the Courts can assess the correct fee which applies, each party will need to complete a Party Category Information Form.
