--- previous+++ current@@ -1,7 +1,8 @@ In some cases you can make an official complaint against an IP attorney.
 When a complaint can be made
 The Trans-Tasman IP Attorneys Board investigates complaints against attorneys.
-A complaint can be made where an attorney:
+ 
+ A complaint can be made where an attorney:
 - Has breached the Code of Conduct
 - Has acted in a way that falls short of the standard of competence, diligence and behaviour expected of an attorney
 - Has acted in a way that shows they are not of good fame, integrity and character
@@ -13,7 +14,8 @@ - The public interest
 - The interests of the profession.
 If the registered attorney is unable to equally comply with all of the obligations, they must prioritise in the order listed above.
-Other key requirements include:
+ 
+ Other key requirements include:
 - Being courteous, ethical and well-informed; not fraudulent, deceitful or knowingly misleading
 - Only do work the attorney is competent to undertake
 - Work in a timely manner
@@ -21,16 +23,21 @@ - Avoid creating an actual or possible conflict of interest between the client's interests and the attorney's interests or the interests of another client.
 When a complaint can't be made
 Complaints can only be made against registered Australian trade marks attorneys or registered Trans-Tasman patent attorneys. You can check whether someone is registered in our directory.
-Search for an attorney
-If the subject of your complaint is not a registered attorney but is a lawyer, you should contact the relevant law society where they practise.
+ 
+ Search for an attorney
+ 
+ If the subject of your complaint is not a registered attorney but is a lawyer, you should contact the relevant law society where they practise.
 Complaint process and outcomes
 The board investigates complaints to determine whether it is likely that the attorney has committed an offence against the patent or trade marks attorney regulations.
-The board refers a complaint to the disciplinary tribunal if they think it is likely that the attorney will be found guilty.
-The tribunal decides:
+ 
+ The board refers a complaint to the disciplinary tribunal if they think it is likely that the attorney will be found guilty.
+ 
+ The tribunal decides:
 - If the attorney has committed an offence
 - Whether to impose a penalty.
 Penalties include suspending or cancelling an attorney's registration or requiring additional training or supervision.
-The board's disciplinary processes are intended to protect the public from further harm. The board and the tribunal can't:
+ 
+ The board's disciplinary processes are intended to protect the public from further harm. The board and the tribunal can't:
 - Order an attorney to pay damages or compensation
 - Force an attorney to do anything to resolve an issue.
 If you are seeking compensation, you will need to consider other avenues such as court action. Some professional organisations, such as the Institute of Patent and Trade Mark Attorneys of Australia, may provide alternatives for complaints against their members.
