--- previous+++ current@@ -4,10 +4,6 @@ How does it work?
 Here's a quick video that explains how to commercialise your IP.
 What is commercialisation?
-If you have a promising idea or technology, commercialising it will help you make a profit or generate value.
-You can generate the profit yourself by launching the product independently, or you can work with others to bring the product to market. This may involve assigning, licensing, or franchising your intellectual property, or IP for short.
-Before commercialising your idea, it’s important to secure intellectual property rights in markets that you want to operate, so you can protect it from imitation and prevent competitors from using it in their own commercial activities.
-Want to learn more? Check out our Commercialise my IP page.
 What to consider before commercialising your IP
 Here's some things to think about before you get started.
 Secure IP rights before you go to market
@@ -68,26 +64,6 @@ Return
 The level of return is high if you aren't sharing profit, but you may have limited ability to scale.
 How it works
-You can launch your new product or service independently, and establish your own market presence.
-You might outsource some manufacturing but otherwise, you take on the work, the risk and the benefits associated with launching a new product.
-Commercialising independently might involve:
-- Creating value for under-served customers
-- Developing or acquiring the capabilities to manufacture and market your innovations
-- Building out the standards and supply chain for your industry.
-This might be a good option if your idea:
-- Has the potential to displace existing products
-- Was created in-house and the business has, or can gain, the capabilities to bring it to market.
-Do I still own the IP?
-Yes, and you'll continue to be very involved in the IP's journey.
-Do I have to pay for the IP?
-You'll need to pay for the cost of developing the IP and the future costs of protecting it, such as renewal fees.
-When will I start to make money?
-You'll see a financial return once you start selling your product or delivering your service.
-Risk
-There's maximum risk involved, as you're investing in the production of the product or service. If there aren't any sales, you won't see a profit.
-Return
-The level of return is high if you aren't sharing profit, but you may have limited ability to scale.
-How it works
 Licensing, rather than selling, is a common commercialisation option. This means giving permission for someone else to use your IP on agreed terms and conditions, through one or more licensing agreements.
 Licensing can be an effective strategy when you don’t have the resources or experience to develop and market your product or service yourself.
 Typically, each licensee will pay a percentage of their sales revenue to you on an agreed basis.
@@ -101,18 +77,6 @@ The risk level is reduced as you'll have responsibility for some cost of development and delivery, but if there aren't any sales, you won't receive any royalties.
 Return
 While the return is reduced per sale as a sharing profit with the licensee, there's a potential for greater overall profit from more sales.
-How it works
-An assignment is the sale of your IP. This involves you transferring the ownership of the IP to another entity, and therefore losing the rights to use the IP.
-Do I still own the IP?
-No, and you won't have any involvement moving forward.
-Do I have to pay for the IP?
-The assignee will bear the risk of commercial development, and be responsible for all costs, including renewal fees and production expenses.
-When will I start to make money?
-Once you've signed the agreement, you'll receive payment for the sale.
-Risk
-There's minimal risk involved in assigning the IP to someone else. There's no outlay of funds for development or delivery of the idea. Even if there are future sales that come from the IP, you've already received your payment.
-Return
-You'll receive a guaranteed payment regardless of whether any products or services are sold. However, this is usually at a reduced price compared to what you might make by commercialising the idea yourself, or by licensing or franchising your idea.
 How it works
 Licensing, rather than selling, is a common commercialisation option. This means giving permission for someone else to use your IP on agreed terms and conditions, through one or more licensing agreements.
 Licensing can be an effective strategy when you don’t have the resources or experience to develop and market your product or service yourself.
