--- previous+++ current@@ -1,16 +1,26 @@-| 6.13. Representations: Pairs and mirror images | | Minor update to page title. | 
+| 6.13. Representations: Pairs and mirror images | | Minor adjustments to page. | 
+ 
+ | 12.8. Section 43 refusal to register: Integrated circuits | | Page reviewed and information updated. | 
+ 
+ | Welcome | | Reviewed and updated. | 
+ 
+ | 8.3. Examination and certification: Requesting examination | | Minor formatting update. | 
  
  | 8.9. Examination and certification: Time frame for completion of examination and last-minute responses | | Reviewed and updated guidance included. | 
- 
- | 8.3. Examination and certification: Requesting examination | | Reviewed and updated guidance included. | 
  
  | Disclaimer | | Review and update Introduction (Disclaimer). | 
  
  | 6.11. Representations: Different views | | fix one minor image issue | 
  
+ | Glossary | | Republished to remove fixed font-size styles for accessibility | 
+ 
  | 8.15. Examination and certification: Examination hearings | | Republished to remove fixed font-size styles for accessibility | 
  
- | Glossary | | Republished to remove fixed font-size styles for accessibility | 
+ | 6.2. Representations: General requirements | | Accessibility fix - remove empty headings | 
+ 
+ | 6.5. Representations: Consistency | | Accessibility fix - remove empty headings | 
+ 
+ | 2.6. Application for registration: Types of applications (s 22) | | Accessibility fix - remove empty headings | 
  
  | 25.6. Ownership disputes: Typical situations where ownership disputes arise | | Accessibility fix - remove empty headings | 
  
@@ -26,14 +36,4 @@  
  | 20.2. Convention priority: Convention countries | | Accessibility fix - remove empty headings | 
  
- | 25.4. Ownership disputes: Disputes where some designs have been registered or published | | Accessibility fix - remove empty headings | 
- 
- | 26.6. Production of documents under s 61(1): Access in ownership disputes | | Accessibility fix - remove empty headings | 
- 
- | 28.4. Hearings: Interface with court proceedings | | Accessibility fix - remove empty headings | 
- 
- | 6.2. Representations: General requirements | | Accessibility fix - remove empty headings | 
- 
- | 6.5. Representations: Consistency | | Accessibility fix - remove empty headings | 
- 
- | 2.6. Application for registration: Types of applications (s 22) | | Accessibility fix - remove empty headings |+ | 25.4. Ownership disputes: Disputes where some designs have been registered or published | | Accessibility fix - remove empty headings |