--- previous+++ current@@ -40,6 +40,12 @@ |
 | 8.15. Examination and certification: Examination hearings | Republished to remove fixed font-size styles for accessibility |
 |
+| 2.7. Application for registration: Request for registration | Accessibility fix - remove empty headings |
+|
+| 28.2. Hearings: Filing evidence | Accessibility fix - remove empty headings |
+|
+| 24.2. Assignments and other interests: Recording changes of ownership | Accessibility fix - remove empty headings |
+|
 | 27.5. Publication and file access: Prohibition orders | Accessibility fix - remove empty headings |
 |
 | 20.2. Convention priority: Convention countries | Accessibility fix - remove empty headings |
@@ -60,18 +66,12 @@ |
 | 25.7. Ownership disputes: Revocation after an ownership dispute | Accessibility fix - remove empty headings |
 |
-| 2.7. Application for registration: Request for registration | Accessibility fix - remove empty headings |
-|
-| 28.2. Hearings: Filing evidence | Accessibility fix - remove empty headings |
-|
-| 24.2. Assignments and other interests: Recording changes of ownership | Accessibility fix - remove empty headings |
-|
 | 11.4. Excluded designs: Amendments | Changed URL from"Designs-that-cannot-be-excluded" |
 |
 | 11.3. Excluded designs: Extensions of time | Updated URL from "Checking-the-priority-status-of-an-excluded-design" to "excluded-designs-extension-of-time" |
 |
 | 11. Excluded designs | Hyperlink now stretches across the complete title and URL has been updated to reflect the changes approved earlier |
 |
-| 19.6. Priority date: Excluded designs | Remove requirements and redirect to Part 11. |
+| 11.1. Excluded designs: overview | Updated to include detailed requirements for an excluded design. |
 |
-| 11.2. Excluded designs: Priority date of excluded designs | Reviewed with significant changes to wording. |+| 19.6. Priority date: Excluded designs | Remove requirements and redirect to Part 11. |