--- previous+++ current@@ -1,7 +1,6 @@ - Home
 - Part 1 Introduction, Quality
-- Part 2 General Filing Requirements
-- Part 3 Filing Requirements for a Trade Mark Application
+- Part 2 Filing Requirements
 - Part 4 Fees
 - Part 5 Data Capture and Indexing
 - Part 6 Expedited Examination
@@ -57,6 +56,12 @@ Recent updates
 | Item amended | Date Amended | Amended Reason | 
 |---|---|---|
+| Part 15.7. Review of the Registrar's decision | Minor text edits | |
+| Part 2.4. Filing Process (excluding filing of applications for registration) | Archiving old content | |
+| Part 2.4 Non-compliance with filing requirements for filing application, notice or request | The Trade Marks Manual of Practice and Procedure has been updated with a new Part 2 – Filing Requirements chapter. Content previously contained in the General Filing Requirements and Filing Requirements for a Trade Mark Application chapters has now been consolidated into this chapter. For further information, refer to the official notices on the IP Australia website. | |
+| Part 2.3 Who may apply, and filing requirements for trade mark applications, notices and requests | The Trade Marks Manual of Practice and Procedure has been updated with a new Part 2 – Filing Requirements chapter. Content previously contained in the General Filing Requirements and Filing Requirements for a Trade Mark Application chapters has now been consolidated into this chapter. For further information, refer to the official notices on the IP Australia website. | |
+| Part 2.2 Introduction to Filing an Application, Notice or Request | The Trade Marks Manual of Practice and Procedure has been updated with a new Part 2 – Filing Requirements chapter. Content previously contained in the General Filing Requirements and Filing Requirements for a Trade Mark Application chapters has now been consolidated into this chapter. For further information, refer to the official notices on the IP Australia website. | |
+| Part 2.1 Landing Page | The General Filing Requirements and Filing Requirements for a Trade Mark Application chapters have been consolidated into a new Part 2 Filing Requirements chapter. For more information, please see the official notices page on IP Australia’s website. | |
 | Part 28. Annex A1 - Information sheet for trade mark applicants | Reviewed and updated following Zip Co Limited v Firstmac Limited [2026] HCA 16. | |
 | Part 9.4. Amendment after particulars of an application have been published (Sections 63, 65 and 65A) | Outdated terms updated. | |
 | 28.3. Examining evidence of honest concurrent use - the five criteria | Reviewed and updated following Zip Co Limited v Firstmac Limited [2026] HCA 16. | |
@@ -70,10 +75,4 @@ | 13.8. Divisional Applications from Series | Whole part reviewed and content updated, including examples. | |
 | 13.1. Series of Trade Marks - Act | Whole part reviewed and content updated, including examples. | |
 | 13.5. Paragraph 51(1)(a) Statements or Representations as to Goods or Services | Whole part reviewed and content updated, including examples. | |
-| 13.2. Material Particulars | Whole part reviewed and content updated, including examples. | |
-| 13.4. Applying Requirements for Material Particulars and Provisions of Paragraphs 51(1)(a), (b) and (c) | Whole part reviewed and content updated, including examples. | |
-| Part 34.7. Evidence required for defensive applications | Part 34.7 reviewed and updated, including references to recent case law and hearing decisions. | |
-| 14. Annex A13 List of terms too broad for classification | Minor updates to suggested alternatives to reflect Nice 13 changes. | |
-| 21.3. Shape (three-dimensional) trade marks | Part 21.3 reviewed, content updated, including reference to Koninklijke Douwe Egberts BV v Cantarella Bros Pty Ltd [2024] FCA 1277. | |
-| 60.4.7 Examination of an IRDA | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. | |
-| 32A.2.1 Section 42: Contrary to Law | Content updated - references to s 27 of the PBR Act removed |+| 13.2. Material Particulars | Whole part reviewed and content updated, including examples. |