--- previous+++ current@@ -57,28 +57,6 @@ Recent updates
 | Item amended | Date Amended | Amended Reason |
 |---|---|---|
-| 60.4.3 Filing/Data Capture/Allocation of Australian Trade Mark Number | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.6 Classification of Goods and Services | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.9 Provisional Refusal | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.14 Acceptance | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.16 Extension of Time to File Notice of Opposition to Protection | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.21 Opposition to Cessation of Protection because of Non-Use | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.19 Cessation or Limitation of Protection | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60. Glossary | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.26 Replacement | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.2.4 Certifying Process | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.3. The Basic Application or Basic Registration (Basic Trade Mark) | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
 | 60.4.2 Record of International Registrations | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
 |
 | 60.4.4 Indexing | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
@@ -95,4 +73,26 @@ |
 | 60.4.23 Claim to Interest in, or Right in Respect of a Trade Mark | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
 |
-| 60.2.2 International Application Form | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |+| 60.2.2 International Application Form | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.24 Change in Ownership of an International Registration | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.2.3 Data Entry | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.1 General Description | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.2.6 Renewal | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.5 Expedite | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.8 Reporting on an IRDA | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.12 Extension of Time | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.15 Revocation of Acceptance | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.18 Protection | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.22 Renewal | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60. Landing Page | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |