--- previous+++ current@@ -59,9 +59,23 @@ |---|---|---|
 | 60.4.7 Examination of an IRDA | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
 |
+| 32A. Annex 1 - Applicable Sections of the PBR Act | Content updated - references to s 27 of the PBR Act removed |
+|
 | 32A.2.1 Section 42: Contrary to Law | Content updated - references to s 27 of the PBR Act removed |
 |
-| 32A. Annex 1 - Applicable Sections of the PBR Act | Content updated - references to s 27 of the PBR Act removed |
+| 60.4.23 Claim to Interest in, or Right in Respect of a Trade Mark | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.2.2 International Application Form | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.24 Change in Ownership of an International Registration | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.2.3 Data Entry | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.1 General Description | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.2.6 Renewal | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
+|
+| 60.4.5 Expedite | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
 |
 | 60.4.8 Reporting on an IRDA | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
 |
@@ -81,18 +95,4 @@ |
 | 60.4.27 Customs | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
 |
-| 60.2.5 Fees for International Applications | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.2.1 General Description | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.3 Filing/Data Capture/Allocation of Australian Trade Mark Number | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.6 Classification of Goods and Services | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.9 Provisional Refusal | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.14 Acceptance | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.16 Extension of Time to File Notice of Opposition to Protection | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |
-|
-| 60.4.21 Opposition to Cessation of Protection because of Non-Use | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |+| 60.2.5 Fees for International Applications | Part reviewed: Minor content updates, links to legislation updated, minor typographical corrections. |