--- previous+++ current@@ -1,8 +1,4 @@ | 7. Can the divisional details be deleted from a valid divisional application? | | Part reviewed, content updated. | 
- 
- | 31.1. Prohibited signs - subsection 39(1) | | Whole part reviewed, content updated. | 
- 
- | 31. Relevant Legislation | | Whole part reviewed, content updated. | 
  
  | 31.2. Prescribed signs - subsection 39(2) | | Whole part reviewed, content updated. | 
  
@@ -12,15 +8,13 @@  
  | 31.4. Practice for examining trade marks containing the signs specified in reg 4.15 | | Whole part reviewed, content updated. | 
  
+ | 31.1. Prohibited signs - subsection 39(1) | | Whole part reviewed, content updated. | 
+ 
+ | 31. Relevant Legislation | | Whole part reviewed, content updated. | 
+ 
  | 30. Annex A2 - Official notice re copyright in the Aboriginal Flag | | Archived as content is outdated and no longer required. | 
  
  | 30.3. Use contrary to law | | Updated content at 3.4.1 in relation to copyright in the Aboriginal and Torres Strait Islander flags. | 
- 
- | 2. Why file a divisional application? | | Part reviewed, content updated. | 
- 
- | 9. Divisional applications and the Intellectual Property Laws Amendment (Raising the Bar) Act 2012 | | Part reviewed, content updated. | 
- 
- | 6. Can a divisional application be based on a parent application which is itself a divisional application? What is the filing date in this situation? | | Part reviewed, content updated. | 
  
  | 12.1. Divisional applications - general | | Part reviewed, content updated. | 
  
@@ -36,4 +30,10 @@  
  | 10. Divisionals and classification | | Part reviewed, content updated. | 
  
- | 3. Conditions for a valid divisional application | | Part reviewed, content updated. |+ | 3. Conditions for a valid divisional application | | Part reviewed, content updated. | 
+ 
+ | 2. Why file a divisional application? | | Part reviewed, content updated. | 
+ 
+ | 9. Divisional applications and the Intellectual Property Laws Amendment (Raising the Bar) Act 2012 | | Part reviewed, content updated. | 
+ 
+ | 6. Can a divisional application be based on a parent application which is itself a divisional application? What is the filing date in this situation? | | Part reviewed, content updated. |