--- previous+++ current@@ -109,7 +109,7 @@ - For a single trademark, enter the required trademark information, then click Connect trademark.
 - For 10 or more trademarks, click Connect multiple trademarks, download the template, enter the required information, then upload your completed template.
 Can Brand Registry help with utility patent disputes?
-Yes. For US brands with a utility patent, Brand Registry offers a neutral evaluation process within the Report a Violation tool. The Amazon Patent Evaluation Express program brings in neutral third-party evaluators who are attorneys skilled in patent analysis, to decide whether an asserted utility patent is infringed.
+Yes. For US brands with a utility patent, Brand Registry offers a neutral evaluation process within the Report a Violation tool. The Amazon Patent Evaluation Express program brings in neutral third-party evaluators who are attorneys skilled in patent analysis, to decide whether an asserted utility patent is infringed. 
 The process is an efficient means for utility patent owners and those accused of patent infringement to address their concerns. It is faster than the court system and is free for the party that the evaluator determines is correct.
 Once a report is submitted, we notify sellers who have listings for reported products so they can participate in an evaluation and defend their claim if they choose.
 Participation in an evaluation requires both parties to deposit $4,000 with the neutral evaluator. Amazon does not handle the deposit and does not retain any portion of the evaluator’s fee. The evaluator reviews submissions and determines whether the reported listings infringe the patent.
@@ -122,18 +122,19 @@ Once an evaluation is complete, if an application meets the criteria, utility patent owners are granted an Amazon Patent Evaluation Express ID associated with this decision. Brands can use the ID from the winning evaluation to report future infringement of that patent through Report a Violation.
 What are Amazon’s policies on intellectual property and brand names?
 How can I report policy violations or other forms of abuse not related to intellectual property?
-You can use the Report abuse tool to report inaccurate listing information, violations of Amazon policy, and other issues. To access the tool, log in to Seller Central and use the main menu to select Performance, then Account Health. On the Account Health page, select Report abuse. If you don't have access to Seller Central, you can report issues using Brand Registry Support. Log in to your Brand Registry account and select Contact us from your homepage, then select Report a marketplace policy violation.
+You can use the Report abuse tool to report inaccurate listing information, violations of Amazon policy, and other issues. To access the tool, log in to Seller Central and use the main menu to select Performance, then Account Health. On the Account Health page, select Report abuse. If you don't have access to Seller Central, you can report issues using Brand Registry Support. Log in to your Brand Registry account and select Contact us from your homepage, then select Report a marketplace policy violation. 
 Amazon also provides a publicly available form for reporting alleged intellectual property infringements such as copyright, trademark, and patent concerns.
 Amazon also provides a publicly available form for reporting alleged intellectual property infringements such as copyright, trademark, and patent concerns.
 The current Administrator has left the company or is no longer associated with the brand. How can I update the Administrator?
 If there are no active Administrators for the brand, there are two ways to request access to the brand:
+ 
 Option 1: If you have at least one other brand that is already enrolled in Brand Registry and you can access the Brand Registry portal, you can submit your request via Brand Registry Support. Select Update brand ownership as the topic. Your request will be reviewed, and you'll receive a response accordingly.
 Option 2: You can attempt to apply for Brand Registry with the same trademark number as a currently enrolled brand. Once you enter the already registered trademark number, a warning will appear, offering a link to a form for sharing your contact information. Select the link to access the Share Contact Information form. At the bottom of the form, there’s a link available only for eligible users, leading to an Appeal Submission form. If your appeal succeeds, you'll be added to the brand with the Administrator and Rights Owner roles.
 Option 1: If you have at least one other brand that is already enrolled in Brand Registry and you can access the Brand Registry portal, you can submit your request via Brand Registry Support. Select Update brand ownership as the topic. Your request will be reviewed, and you'll receive a response accordingly.
 Option 2: You can attempt to apply for Brand Registry with the same trademark number as a currently enrolled brand. Once you enter the already registered trademark number, a warning will appear, offering a link to a form for sharing your contact information. Select the link to access the Share Contact Information form. At the bottom of the form, there’s a link available only for eligible users, leading to an Appeal Submission form. If your appeal succeeds, you'll be added to the brand with the Administrator and Rights Owner roles.
 Launch your brand with Amazon
 Enroll your brand in Amazon Brand Registry to start unlocking a suite of brand-building tools and protection benefits.
-1Amazon internal data
+1Amazon internal data 
 2Amazon internal data, August 2024
 3Amazon internal data, May 2026
 2Amazon internal data, August 2024
