--- previous+++ current@@ -68,7 +68,7 @@ Protect your brand
 Transparency
 Use unique serial codes to stop counterfeits and improve customer engagement with authenticated items.
-Over 2.5 billion product units have been verified as genuine through Amazon's Transparency program.3
+More than 2.7 billion product units have been verified as genuine through Amazon's Transparency program3
 Report a Violation
 Detect and report suspected IP infringement. Each report helps strengthen our automated protections.
 Project Zero
@@ -135,6 +135,6 @@ Enroll your brand in Amazon Brand Registry to start unlocking a suite of brand-building tools and protection benefits.
 1Amazon internal data
 2Amazon internal data, August 2024
-32024 Amazon Brand Protection Report
+3Amazon internal data, May 2026
 2Amazon internal data, August 2024
-32024 Amazon Brand Protection Report+3Amazon internal data, May 2026