--- previous+++ current@@ -2,11 +2,14 @@ - If you choose a company as your business structure, you must register it on the Australian companies register, managed by ASIC.
 - Before registering a company, you will need to have some building blocks in place.
 - Company officeholders have ongoing legal obligations after the company is registered.
+ 
 Before registering a company
 Before registering a company, you should decide if it is the right business structure for you:
 Sole trader? Partnership? Company? Trust?
 Other business structures such as sole traders, partnerships and trusts do not need to register as a company.
 The companies register and the business names register are separate. If you want to operate under a business name that is different to your company name, you will also need to register a business name.
+ 
+ 
 Business.gov.au has a ‘Help me decide’ tool that can help you work out the registrations you need for different business structures.
 Building blocks you need to register a company
 Company name
@@ -43,12 +46,14 @@ - their place and date of birth.
 People will be able to see these details online on the companies register.
 If a proposed officeholder is worried about their safety, they can ask us to hide their residential address from the public. Read this page before you register, and do not register using the BRS process outlined below.
+ 
 Directors must apply for a director ID before a company is registered:
 Director identification number: Australian Business Registry Services
 A company must obtain and keep your written and signed consent before you become a company director or secretary. Here is an example of an officeholder consent template.
 Find out what to do if you did not give consent to be a company officeholder
 For more information on officeholders, see:
 Company officeholders (directors and secretaries)
+ 
 Fees for registering a company
 There is a fee to register a company. For most companies, the current fee is {I01} or {I02}, depending on the type of company.
 See fees for all company types.
