--- previous+++ current@@ -8,11 +8,11 @@ - Provisional schedule for IGC in 2026
 News
 - 
-March 17, 2026 
+IGC 52 Summary March 17, 2026
 - 
-52nd Session of the IGC – March 4 to 13, 2026 January 4, 2026 
+52nd Session of the IGC – March 4 to 13, 2026 January 4, 2026
 - 
-The 2025 General Assembly Renewed the IGC’s Mandate August 4, 2025 
+The 2025 General Assembly Renewed the IGC’s Mandate August 4, 2025
 Featured
 Relevant IGC documents
 - The Protection of Traditional Cultural Expressions: Updated Draft Gap Analysis
@@ -26,7 +26,8 @@ Ongoing and archived consultations of IGC members and accredited observers. Key issues and gap analyses on the protection of traditional cultural expressions and traditional knowledge.
 Email updates
 Traditional Knowledge Updates – Regular news and resources about intellectual property & traditional knowledge; and IGC updates.
-Sign up | All WIPO newslettersResources for participants
+Sign up | All WIPO newsletters
+Resources for participants
 Related links
 IP & other policy topics
 Information resources
