--- previous+++ current@@ -26,7 +26,7 @@ When you have a registered trade mark, you can protect it with the R symbol. It's an offence to use this symbol before it's registered.
 Monitor the market
 Market monitoring is a key part of managing your IP. It takes some effort to keep on top of what's happening in the market, but it's a helpful way for you to stay aware of any potential infringement. If you need help with monitoring the market, you can pay someone to do this for you. Ask your legal representative for service recommendations.
-If you've noticed that someone may be using your IP without your permission, you can take action to stop them.
+If you've noticed that someone may be using your IP without your permission, you can take action to stop them. 
 Search existing IP
 It's a good idea to keep watch of new IP entering the market. You can keep up to date with proposed and accepted IP by regularly checking the:
 - Australian Trade Marks Search
@@ -37,8 +37,11 @@ Counterfeit goods — imitations of your products, often sold at a lower price — are a threat to your business.
 Counterfeit goods can:
 - Harm brand reputation
+ 
 - Impact profits
+ 
 - Discourage innovation
+ 
 - Risk consumer health and safety.
 You should be proactive by listing your IP rights with platforms before anyone tries to sell counterfeit goods. Some platforms invite you to add your trade marks and other kinds of IP rights to a brand registry. This is used in automated checks to detect infringing goods before they are even listed for sale.
 Amazon Brand Registry
