--- previous+++ current@@ -1,4 +1,5 @@ What is WIPO?
+The World Intellectual Property Organization (WIPO) is the United Nations agency that serves the world’s innovators and creators, ensuring that their ideas travel safely to the market and improve lives everywhere.
 The World Intellectual Property Organization (WIPO) is the United Nations agency that serves the world’s innovators and creators, ensuring that their ideas travel safely to the market and improve lives everywhere.
 We do so by providing services that enable creators, innovators and entrepreneurs to protect and promote their intellectual property (IP) across borders and acting as a forum for addressing cutting-edge IP issues. Our IP data and information guide decisionmakers the world over. And our impact-driven projects and technical assistance ensure IP benefits everyone, everywhere.
 Quick facts
@@ -16,7 +17,6 @@ - innovation ecosystems connected with smart digital tools that make knowledge more accessible
 - targeted support that helps all countries unlock IP’s value for growth and development
 - trusted multilingual data and insights that power decisions, policy and progress
-- help Australian small business owners protect their intellectual property with our new OzIP Protection Service
 WIPO Services Directory
 All services and contacts, one directory
 Video: WIPO in Three Minutes
