--- previous+++ current@@ -26,6 +26,13 @@ - how evidence in a Trade Marks Act matter should be filed
 - how to pay fees.
 Here's the current non-legislative trade mark instruments:
+The Designs Act to allows the Registrar to determine:
+- how to file a document with the Designs office
+- forms for filing documents
+- how evidence in a Designs Act matter should be filed
+- formal requirements for design representations
+- how to pay fees.
+Here's the current non-legislative design instruments:
 The Patents Act allows the Commissioner to determine:
 - 
  how to file a document with the patent office
@@ -38,29 +45,6 @@ - 
  formalities requirements for complete applications.
 Here's the current non-legislative patent instruments:
-Patents (Formalities Requirements for Patent Documents) Determination 2022
-PDF | 296KB
-Patents (Approved Means for Paying a Fee) Determination 2026
-PDF | 310KB
-Patents (Means and Form of Filing Documents and Evidence) (Opposition and Other Proceedings) Instrument 2026
-PDF | 316KB
-Patents (Means and Form of Filing Documents) Instrument 2026
-PDF | 310KB
-The Designs Act to allows the Registrar to determine:
-- how to file a document with the Designs office
-- forms for filing documents
-- how evidence in a Designs Act matter should be filed
-- formal requirements for design representations
-- how to pay fees.
-Here's the current non-legislative design instruments:
-Designs (Approved Means for Paying a Fee) Determination 2026
-PDF | 284KB
-Designs (Documents and Evidence) (Proceedings) Instrument 2026
-PDF | 312KB
-Designs (Formal Requirements for Designs Documents) Instrument 2022
-PDF | 498KB
-Designs (Means and Form of Filing Documents) Instrument 2026
-PDF | 242KB
 The Plant Breeder's Rights Act allows the Registrar to determine:
 - how to file a document with the plant breeder's right office
 - forms for filing documents
