--- previous+++ current@@ -1,3 +1,5 @@+- 
+CorporateStrategic Corporate Plan 2026–27 outlines priorities for year 01 September 2026 | 2 MIN READ From artificial intelligence (AI) governance to First Nations engagement, find out IP Australia's key focus areas for the year ahead.
 - 
 Design rights Patents Plant breeders rights Trade marksStartup Muster 2026 survey now open for Australian startups and supporters 17 August 2026 | 2 MIN READ IP Australia is supporting Startup Muster and encouraging Australia's startup community to take part in the nation's largest survey of the startup ecosystem.
 - 
@@ -16,6 +18,4 @@ PatentsThis inventor is turning trash into treasure 23 June 2026 | 8 MIN READ Discover how Professor Veena Sahajwalla AO transformed waste into valuable materials, and how patents supported the commercialisation of her inventions.
 - 
 Trade marksA copycat can't even come close 16 June 2026 | 7 MIN READ Discover how entrepreneur Tess Robinson navigates the complex IP landscape. Learn how to secure your trade marks early and protect your business from copycats.
-- 
-Patents ApplyIP Report shows intellectual property supports productivity and investment in emerging industries 11 June 2026 | 3 MIN READ New research in the Australian IP Report 2026 suggests Australia's role in battery circularity innovation is expanding toward high-value technology for refining, recycling and recovering materials.
  Showing 38 results