--- previous+++ current@@ -1,4 +1,8 @@ -
+Patents Apply Amend
+IP Australia joins PPH eXtra Initiative
+02 June 2026 | 2 MIN READ
+New transparency program gives applicants clearer information on patent examination timeframes for Patent Prosecution Highway requests. -
 Design rights Patents Plant breeders rights Trade marks Copyright Office of Chief Economist
 Australian IP Report 2026 reveals businesses with IP see stronger productivity and income growth
 06 May 2026 | 2 MIN READ
@@ -34,9 +38,5 @@ Patents Apply
 New international search options now available
 01 March 2026 | 3 MIN READ
-Patent applicants can now choose the European Patent Office (EPO) or Intellectual Property Office of Singapore (IPOS) for international search and examination. -
-Other
-Marking the first year of the Indigenous Knowledge Panel
-11 February 2026 | 2 MIN READ
-One year of the Indigenous Knowledge Panel supporting meaningful change.
-Showing 27 results+Patent applicants can now choose the European Patent Office (EPO) or Intellectual Property Office of Singapore (IPOS) for international search and examination.
+Showing 28 results