--- previous+++ current@@ -1,3 +1,5 @@+- 
+Trade marks CopyrightAmid a trade mark battle, Clothing The Gaps founders showed their values proudly 07 July 2026 | 6 MIN READ Clothing The Gaps faced trade mark opposition from The Gap and a copyright dispute over the Aboriginal flag. Find out how they stayed true to their purpose.
 - 
 Design rights Office of Chief Economist Patents Plant breeders rights Trade marksIP Report shows Australian resilience amid structural shifts in the global economy 02 July 2026 The Australian IP Report 2026 shows Australia holding its ground during a period of significant structural change in the global economy, as trade relationships diversify, and competitive pressures reshape international markets.
 - 
@@ -16,6 +18,4 @@ Design rights Patents Plant breeders rights Trade marks Copyright Office of Chief EconomistAustralian IP Report 2026 reveals businesses with IP see stronger productivity and income growth 06 May 2026 | 2 MIN READ The Australian IP Report 2026 has been released, presenting the latest statistics and research on the use of intellectual property (IP) rights in Australia.
 - 
 Design rights Patents Plant breeders rights Trade marksUnderstanding intellectual property issues: Introducing IP First Response’s new chatbot 03 May 2026 | 2 MIN READ A new chatbot feature is helping small businesses understand their IP options.
-- 
-Patents ApplyUpdate to patent excess claims reminder process 01 May 2026 | 3 MIN READ From 1 July 2026, we will issue a 3-month reminder for excess claims after examination requests are filed.
- Showing 34 results+ Showing 35 results