--- previous+++ current@@ -1,4 +1,16 @@ -
+Design rights Patents Plant breeders rights Trade marks
+Understanding intellectual property issues: Introducing IP First Response’s new chatbot
+03 May 2026 | 2 MIN READ
+A new chatbot feature is helping small businesses understand their IP options. -
+Patents Apply
+Update to patent excess claims reminder process
+01 May 2026 | 3 MIN READ
+From 1 July 2026, we will issue a 3-month reminder for excess claims after examination requests are filed. -
+Trade marks Enforce
+Small business, big names: Why protecting your IP early matters
+01 May 2026 | 3 MIN READ
+Increasingly, Australian small businesses and independent creators are finding themselves in IP disputes. -
 Design rights Patents Plant breeders rights Trade marks
 Australia-EU Free Trade Agreement concluded
 24 March 2026 | 1 MIN READ
@@ -26,17 +38,5 @@ Patents Apply
 Pilot arrangements with European Patent Office coming soon
 11 February 2026 | 3 MIN READ
-Australian innovators to benefit from new arrangements with European Patent Office from 1 March. -
-Trade marks Patents Design rights Plant breeders rights
-Introducing The IP Australia Podcast: navigating the world of intellectual property
-09 February 2026 | 4 MIN READ
-Sharing real stories and practical insights to help start-ups and small business understand intellectual property. -
-Corporate
-EPO and IP Australia to launch PCT pilot programme starting 1 March 2026
-19 January 2026 | 2 MIN READ
-Australian innovators will be able to choose the EPO as their International Searching Authority and International Preliminary Examining Authority. -
-Patents Trade marks Design rights Plant breeders rights Corporate
-IP Australia a world-leader for innovation in IP agencies
-12 January 2026 | 3 MIN READ
-World Trademark Review highlights our digital capabilities and customer-focused approach.
-Showing 20 results+Australian innovators to benefit from new arrangements with European Patent Office from 1 March.
+Showing 24 results