--- previous+++ current@@ -1,3 +1,5 @@+- 
+Trade marksThis founder secured her recipe for success 21 July 2026 | 6 MIN READ Janae Henkel put intellectual property in the 'too-hard basket' for years. Learn why this founder says protecting your brand is easier than you think.
 - 
 Copyright Trade marksThe Digital Picnic founder knew the value of her IP from day dot 14 July 2026 | 5 MIN READ A cease and desist letter forced Cherie Clonan to rethink her business. Learn why protecting your intellectual property early can save time, money and stress.
 - 
@@ -16,6 +18,4 @@ Patents ApplyIP Report shows intellectual property supports productivity and investment in emerging industries 11 June 2026 | 3 MIN READ New research in the Australian IP Report 2026 suggests Australia's role in battery circularity innovation is expanding toward high-value technology for refining, recycling and recovering materials.
 - 
 Patents Apply AmendIP Australia joins PPH eXtra Initiative 02 June 2026 | 2 MIN READ New transparency program gives applicants clearer information on patent examination timeframes for Patent Prosecution Highway requests.
-- 
-Design rights Patents Plant breeders rights Trade marks Copyright Office of Chief EconomistAustralian IP Report 2026 reveals businesses with IP see stronger productivity and income growth 06 May 2026 | 2 MIN READ The Australian IP Report 2026 has been released, presenting the latest statistics and research on the use of intellectual property (IP) rights in Australia.
- Showing 36 results+ Showing 37 results