--- previous+++ current@@ -1,3 +1,5 @@+- 
+Design rightsA unique design sets you apart 26 June 2026 | 5 MIN READ Protect the look of your business and product's distinctive design. Learn how design rights help you innovate.
 - 
 PatentsThis inventor is turning trash into treasure 23 June 2026 | 8 MIN READ Discover how Professor Veena Sahajwalla AO transformed waste into valuable materials, and how patents supported the commercialisation of her inventions.
 - 
@@ -16,6 +18,4 @@ Trade marks EnforceSmall business, big names: Why protecting your IP early matters 01 May 2026 | 3 MIN READ Increasingly, Australian small businesses and independent creators are finding themselves in IP disputes.
 - 
 Design rights Patents Plant breeders rights Trade marksAustralia-EU Free Trade Agreement concluded 24 March 2026 | 1 MIN READ Intellectual property chapter provides for the protection of geographical indications.
-- 
-Design rights Patents Plant breeders rights Trade marksHave your say on proposals to streamline and simplify IP legislation 10 March 2026 | 2 MIN READ Public consultation is now open on proposals to reduce regulatory burdens and complexity across the IP rights.
- Showing 31 results+ Showing 32 results