--- previous+++ current@@ -1,3 +1,7 @@+- 
+Design rights Office of Chief Economist Patents Plant breeders rights Trade marksIP Report shows Australian resilience amid structural shifts in the global economy 02 July 2026 The Australian IP Report 2026 shows Australia holding its ground during a period of significant structural change in the global economy, as trade relationships diversify, and competitive pressures reshape international markets.
+- 
+PatentsThis medical pioneer got patenting right the second time around 30 June 2026 | 7 MIN READ Learn how medical pioneer Professor Fiona Wood successfully patented spray-on skin technology. Gain insights into IP and commercialisation to protect your ideas.
 - 
 Design rightsA unique design sets you apart 26 June 2026 | 5 MIN READ Protect the look of your business and product's distinctive design. Learn how design rights help you innovate.
 - 
@@ -14,8 +18,4 @@ Design rights Patents Plant breeders rights Trade marksUnderstanding intellectual property issues: Introducing IP First Response’s new chatbot 03 May 2026 | 2 MIN READ A new chatbot feature is helping small businesses understand their IP options.
 - 
 Patents ApplyUpdate to patent excess claims reminder process 01 May 2026 | 3 MIN READ From 1 July 2026, we will issue a 3-month reminder for excess claims after examination requests are filed.
-- 
-Trade marks EnforceSmall business, big names: Why protecting your IP early matters 01 May 2026 | 3 MIN READ Increasingly, Australian small businesses and independent creators are finding themselves in IP disputes.
-- 
-Design rights Patents Plant breeders rights Trade marksAustralia-EU Free Trade Agreement concluded 24 March 2026 | 1 MIN READ Intellectual property chapter provides for the protection of geographical indications.
- Showing 32 results+ Showing 34 results