--- previous+++ current@@ -1,4 +1,8 @@ -
+Design rights Patents Plant breeders rights Trade marks Copyright Office of Chief Economist
+Australian IP Report 2026 reveals businesses with IP see stronger productivity and income growth
+06 May 2026 | 2 MIN READ
+The Australian IP Report 2026 has been released, presenting the latest statistics and research on the use of intellectual property (IP) rights in Australia. -
 Design rights Patents Plant breeders rights Trade marks
 Understanding intellectual property issues: Introducing IP First Response’s new chatbot
 03 May 2026 | 2 MIN READ
@@ -34,9 +38,5 @@ Other
 Marking the first year of the Indigenous Knowledge Panel
 11 February 2026 | 2 MIN READ
-One year of the Indigenous Knowledge Panel supporting meaningful change. -
-Patents Apply
-Pilot arrangements with European Patent Office coming soon
-11 February 2026 | 3 MIN READ
-Australian innovators to benefit from new arrangements with European Patent Office from 1 March.
+One year of the Indigenous Knowledge Panel supporting meaningful change.
 Showing 24 results